Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-05-03 07:40:34.995494329 -0700
+++ /tmp/foob	2016-05-03 07:40:34.995494329 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc5+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Apr 24 16:36:06 PDT 2016
+Linux version 4.6.0-rc6+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue May 3 07:35:35 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -94,7 +94,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.652:1): initialized
+audit: type=2000 audit(0.552:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -154,7 +154,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-04-24 23:36:37 UTC (1461540997)
+rtc-pl031 9010000.pl031: setting system clock to 2016-05-03 14:36:05 UTC (1462286165)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-03 07:40:35.015494331 -0700
+++ /tmp/foob	2016-05-03 07:40:35.015494331 -0700
@@ -39,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 #################
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 6918656 (699200 hex)
+Bytes transferred = 6919032 (699378 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x699200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x699378 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc5-82962-g02da2d7 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Apr 24 16:34:30 PDT 2016
+Linux version 4.6.0-rc6-83187-g04974df (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue May 3 07:33:44 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -298,7 +298,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-17 10:55:40 UTC (950784940)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-26 01:55:15 UTC (951530115)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -310,4 +310,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.229387] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.229304] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-03 07:40:35.023494330 -0700
+++ /tmp/foob	2016-05-03 07:40:35.023494330 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -27,6 +27,7 @@
 Tegra114 (Dalmore) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash'
 Tegra114 (Dalmore) # setenv ipaddr 192.168.100.100; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra114 (Dalmore) # tftpboot 0x81000000 zImage.tegra_defconfig
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'zImage.tegra_defconfig'.
@@ -37,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5316440 (511f58 hex)
+Bytes transferred = 5316424 (511f48 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +51,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x511f58 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x511f48 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc5-82962-g02da2d7 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Apr 24 16:34:25 PDT 2016
+Linux version 4.6.0-rc6-83187-g04974df (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue May 3 07:33:39 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5c400, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5c440, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -90,9 +91,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2288   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab22a8   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
+      .data : 0xc0c00000 - 0xc0c5dfc0   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a04   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -295,7 +296,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-17 10:53:04 UTC (950784784)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-26 01:52:47 UTC (951529967)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-03 07:40:35.039494332 -0700
+++ /tmp/foob	2016-05-03 07:40:35.043494332 -0700
@@ -34,9 +34,9 @@
 	 #################################################################
 	 #################################################################
 	 #################
-	 4 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 6918656 (699200 hex)
+Bytes transferred = 6919032 (699378 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -47,7 +47,7 @@
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x699200 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x699378 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc5-82962-g02da2d7 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Apr 24 16:34:30 PDT 2016
+Linux version 4.6.0-rc6-83187-g04974df (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue May 3 07:33:44 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -282,9 +282,12 @@
 +1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
 mmc0: new high speed MMC card at address 0001
-tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1
+tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
@@ -338,7 +341,7 @@
 pcie_pme 0000:00:02.0:pcie01: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a12000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 391
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a1c000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 391
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +5V_HDMI_CON: supplied by +5V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
@@ -346,12 +349,8 @@
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-drm drm: No connectors reported connected with modes
-[drm] Cannot find any crtc or sizes - going 1024x768
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1
-Console: switching to colour frame buffer device 128x48
+tegra-hdmi 54280000.hdmi: cannot set audio to 44100 at 297000000 pclk
+Console: switching to colour frame buffer device 512x135
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 +5V_SATA: supplied by +5V_SYS
@@ -380,7 +379,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:03 UTC (946684923)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -395,8 +394,8 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-mmc1: new high speed SDHC card at address aaaa
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
+mmc1: new high speed SDHC card at address aaaa
 mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
  mmcblk1: p1 p2 p3
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-03 07:40:35.047494331 -0700
+++ /tmp/foob	2016-05-03 07:40:35.051494332 -0700
@@ -34,7 +34,7 @@
 	 ######################################
 	 4 MiB/s
 done
-Bytes transferred = 5316440 (511f58 hex)
+Bytes transferred = 5316424 (511f48 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -45,7 +45,7 @@
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x511f58 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x511f48 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc5-82962-g02da2d7 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Apr 24 16:34:25 PDT 2016
+Linux version 4.6.0-rc6-83187-g04974df (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue May 3 07:33:39 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5c400, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5c440, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -85,9 +85,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2288   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab22a8   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
+      .data : 0xc0c00000 - 0xc0c5dfc0   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a04   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-05-03 07:40:35.055494333 -0700
+++ /tmp/foob	2016-05-03 07:40:35.059494333 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800276 us
+Bootloader-Cpu Init at (time stamp): 1800835 us
 
 Pinmux changes applied in kernel way
 
@@ -59,25 +59,25 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 4 us
+Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800249  us
-Bootloader (AosInit) time: 733441 us
-Bootloader Main Init time: 180214 us
-BlockDevMgrInit time: 157550 us
+Miscellaneous avp time: 1800809  us
+Bootloader (AosInit) time: 734226 us
+Bootloader Main Init time: 183569 us
+BlockDevMgrInit time: 157577 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
+StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 77021 us 
-Display init time: 574640 us
-Kernel read time: 308 msec
-Load Kernel Size:   13305856 Bytes
-Bl Kernel ReadSpeed: 43200000 Bytes/sec
-Pre-Kernel Jump time: 1507972 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5523004 us
+Partition Table load: 77010 us 
+Display init time: 574523 us
+Kernel read time: 325 msec
+Load Kernel Size:   14102528 Bytes
+Bl Kernel ReadSpeed: 43392000 Bytes/sec
+Pre-Kernel Jump time: 1571372 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5611201 us
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc5+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Apr 24 16:36:06 PDT 2016
+Linux version 4.6.0-rc6+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue May 3 07:35:35 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -92,7 +92,7 @@
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
 software IO TLB [mem 0xe5200000-0xe9200000] (64MB) mapped at [ffffffc065200000-ffffffc0691fffff]
-Memory: 1643744K/1775616K available (6996K kernel code, 625K rwdata, 2956K rodata, 816K init, 244K bss, 115488K reserved, 16384K cma-reserved)
+Memory: 1642968K/1775616K available (6996K kernel code, 625K rwdata, 2956K rodata, 816K init, 244K bss, 116264K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbdbfff0000   (   246 GB)
@@ -175,11 +175,11 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 1588K (ffffffc001680000 - ffffffc00180d000)
+Freeing initrd memory: 2364K (ffffffc001680000 - ffffffc0018cf000)
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.835:1): initialized
+audit: type=2000 audit(1.051:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -260,23 +260,23 @@
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
+mmc1: Unknown controller version (3). You may experience problems.
 
 System initialization...
 
-Mounting /proc :[    2.578239] sdhci-tegra 700b0400.sdhci: Got CD GPIO
-[SUCCESS]
-Mounting /sys :[    2.589314] sdhci-tegra 700b0400.sdhci: Got WP GPIO
-[SUCCESS]
-Mounting /dev :[    2.599153] mmc1: Unknown controller version (3). You may experience problems.
-[SUCCESS]
+Mounting /proc :[SUCCESS]
+Mounting /sys :[SUCCESS]
+Mounting /dev :[SUCCESS]
 Mounting /dev/pts :[SUCCESS]
 Mounting /sys/kernel/debug :[SUCCESS]
 Mounting other filesystems :[SUCCESS]
 Enabling hot-plug :[SUCCESS]
+Populating /dev :[SUCCESS]
 Starting syslogd :[SUCCESS]
+Starting telnetd :[SUCCESS]
 
 System initialization complete.
 
-
-
 Please press Enter to activate this console. 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-03 07:40:35.067494333 -0700
+++ /tmp/foob	2016-05-03 07:40:35.071494333 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -47,20 +47,20 @@
 	 #################################################################
 	 #################################################################
 	 #################
-	 4.1 MiB/s
+	 905.3 KiB/s
 done
-Bytes transferred = 6918656 (699200 hex)
+Bytes transferred = 6919032 (699378 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 859.4 KiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x699200 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x699378 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc5-82962-g02da2d7 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Apr 24 16:34:30 PDT 2016
+Linux version 4.6.0-rc6-83187-g04974df (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue May 3 07:33:44 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -325,24 +325,24 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-29 13:56:19 UTC (1419861379)
-Waiting for root device PARTUUID=11eb23c1-01...
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
  mmcblk0: p1
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
+asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-03 07:40:35.079494334 -0700
+++ /tmp/foob	2016-05-03 07:40:35.083494333 -0700
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 900.4 KiB/s
+	 907.2 KiB/s
 done
-Bytes transferred = 5316440 (511f58 hex)
+Bytes transferred = 5316424 (511f48 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x511f58 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x511f48 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc5-82962-g02da2d7 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Apr 24 16:34:25 PDT 2016
+Linux version 4.6.0-rc6-83187-g04974df (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue May 3 07:33:39 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0c5c400, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0c5c440, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -97,9 +97,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2288   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab22a8   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
+      .data : 0xc0c00000 - 0xc0c5dfc0   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a04   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -304,17 +304,17 @@
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
+mmc0: host does not support reading read-only switch, assuming write-enable
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 NET: Registered protocol family 10
-mmc0: host does not support reading read-only switch, assuming write-enable
+mip6: Mobile IPv6
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
-mip6: Mobile IPv6
 mmc1: new high speed SD card at address b368
-sit: IPv6 over IPv4 tunneling driver
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
+sit: IPv6 over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -333,23 +333,24 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-hub 1-1:1.0: USB hub found
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-29 13:54:52 UTC (1419861292)
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-05-03 07:40:35.107494335 -0700
+++ /tmp/foob	2016-05-03 07:40:35.111494336 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 730169 us
-Basic modules init took 348715 us
-Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14916 us
-Carveout took 252723 us
-CPU initialization took 92319 us
-Total time taken by TegraBoot 708673 us
+Tegraboot started after 846458 us
+Basic modules init took 348559 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14366 us
+Carveout took 252722 us
+CPU initialization took 92301 us
+Total time taken by TegraBoot 707948 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.570] LPDDR4 Training: Number of tables = 10
+64b[0001.683] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-11674624 bytes read in 739 ms (15.1 MiB/s)
+11674624 bytes read in 724 ms (15.4 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-37136 bytes read in 453 ms (79.1 KiB/s)
+37136 bytes read in 454 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 496 ms (3.1 MiB/s)
+1627787 bytes read in 494 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc5+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Apr 24 16:36:06 PDT 2016
+Linux version 4.6.0-rc6+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue May 3 07:35:35 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -259,7 +259,7 @@
 kvm [1]: timer IRQ3
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.363:1): initialized
+audit: type=2000 audit(0.359:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-03 07:40:35.115494336 -0700
+++ /tmp/foob	2016-05-03 07:40:35.119494336 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #################
-	 3.4 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 6918656 (699200 hex)
+Bytes transferred = 6919032 (699378 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.5 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x699200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x699378 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc5-82962-g02da2d7 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Apr 24 16:34:30 PDT 2016
+Linux version 4.6.0-rc6-83187-g04974df (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue May 3 07:33:44 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -283,8 +283,8 @@
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
@@ -360,7 +360,6 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
-bash: cannot set terminal process group (-1): Inappropriate ioctl for de[    3.713572] usb 2-1: new high-speed USB device number 2 using tegra-ehci
-vice
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    3.708939] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-03 07:40:35.143494337 -0700
+++ /tmp/foob	2016-05-03 07:40:35.147494337 -0700
@@ -39,7 +39,7 @@
 	 ######################################
 	 3.3 MiB/s
 done
-Bytes transferred = 5316440 (511f58 hex)
+Bytes transferred = 5316424 (511f48 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x511f58 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x511f48 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc5-82962-g02da2d7 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Apr 24 16:34:25 PDT 2016
+Linux version 4.6.0-rc6-83187-g04974df (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue May 3 07:33:39 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5c400, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5c440, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -89,9 +89,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2288   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab22a8   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
+      .data : 0xc0c00000 - 0xc0c5dfc0   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a04   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -360,7 +360,7 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
-bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-ba[    3.990608] usb 2-1: new high-speed USB device number 2 using tegra-ehci
-sh: no job control in this shell
+bash: cannot set terminal process group [    3.990603] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+(-1): Inappropriate ioctl for device
+bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
