Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-09-24 17:13:49.842381145 -0700
+++ /tmp/foob	2017-09-24 17:13:49.842381145 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-199401-g2bd6bf0 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sat Sep 16 16:42:19 PDT 2017
+Linux version 4.14.0-rc2-199764-ge19b205 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Sep 24 17:10:07 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
@@ -29,15 +29,15 @@
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 999040K/1048576K available (9212K kernel code, 1142K rwdata, 4568K rodata, 1088K init, 402K bss, 33152K reserved, 16384K cma-reserved)
+Memory: 998976K/1048576K available (9212K kernel code, 1142K rwdata, 4572K rodata, 1152K init, 402K bss, 33216K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
     .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
-      .init : 0xffff000008e00000 - 0xffff000008f10000   (  1088 KB)
-      .data : 0xffff000008f10000 - 0xffff00000902da00   (  1143 KB)
-       .bss : 0xffff00000902da00 - 0xffff000009092470   (   403 KB)
+      .init : 0xffff000008e00000 - 0xffff000008f20000   (  1152 KB)
+      .data : 0xffff000008f20000 - 0xffff00000903da00   (  1143 KB)
+       .bss : 0xffff00000903da00 - 0xffff0000090a2470   (   403 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -77,7 +77,7 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f15000)
+vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f25000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -114,7 +114,7 @@
 hw perfevents: enabled with armv8_pmuv3 PMU driver, 1 counters available
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.940:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.804:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -179,7 +179,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-09-16 23:43:01 UTC (1505605381)
+rtc-pl031 9010000.pl031: setting system clock to 2017-09-25 00:10:51 UTC (1506298251)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
@@ -188,7 +188,7 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 1088K
+Freeing unused kernel memory: 1152K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-09-24 17:13:49.862381229 -0700
+++ /tmp/foob	2017-09-24 17:13:49.862381229 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801888 us
+Bootloader-Cpu Init at (time stamp): 1801362 us
 
 Pinmux changes applied in kernel way
 
@@ -59,25 +59,25 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 3 us
+Board Id Read Time: 4 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801862  us
-Bootloader (AosInit) time: 734248 us
-Bootloader Main Init time: 180084 us
-BlockDevMgrInit time: 157536 us
+Miscellaneous avp time: 1801335  us
+Bootloader (AosInit) time: 733435 us
+Bootloader Main Init time: 180426 us
+BlockDevMgrInit time: 157678 us
 PartMgrInit time: 8 us
-StorMgrInit time: 30 us
-FsMgrInit time: 3 us
-Partition Table load: 76681 us 
-Display init time: 574762 us
-Kernel read time: 1339 msec
-Load Kernel Size:   17760256 Bytes
+StorMgrInit time: 31 us
+FsMgrInit time: 2 us
+Partition Table load: 76958 us 
+Display init time: 574514 us
+Kernel read time: 1344 msec
+Load Kernel Size:   17825792 Bytes
 Bl Kernel ReadSpeed: 13263000 Bytes/sec
-Pre-Kernel Jump time: 3827558 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 8788706 us
+Pre-Kernel Jump time: 3837183 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 8800237 us
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-199401-g2bd6bf0 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sat Sep 16 16:42:19 PDT 2017
+Linux version 4.14.0-rc2-199764-ge19b205 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Sep 24 17:10:07 PDT 2017
 Boot CPU: AArch64 Processor [4e0f0000]
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
@@ -102,15 +102,15 @@
 Policy zone: DMA
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=192511 modem_id=0 android.kerneltype=normal
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 1708264K/1775616K available (9212K kernel code, 1142K rwdata, 4568K rodata, 1088K init, 402K bss, 50968K reserved, 16384K cma-reserved)
+Memory: 1708200K/1775616K available (9212K kernel code, 1142K rwdata, 4572K rodata, 1152K init, 402K bss, 51032K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
     .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
-      .init : 0xffff000008e00000 - 0xffff000008f10000   (  1088 KB)
-      .data : 0xffff000008f10000 - 0xffff00000902da00   (  1143 KB)
-       .bss : 0xffff00000902da00 - 0xffff000009092470   (   403 KB)
+      .init : 0xffff000008e00000 - 0xffff000008f20000   (  1152 KB)
+      .data : 0xffff000008f20000 - 0xffff00000903da00   (  1143 KB)
+       .bss : 0xffff00000903da00 - 0xffff0000090a2470   (   403 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -150,7 +150,7 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f15000)
+vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f25000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -271,16 +271,17 @@
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +VDD_LED: supplied by +VDD_MUX
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
 hctosys: unable to open rtc device (rtc0)
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
 +5V_USB_HS: disabling
@@ -289,8 +290,7 @@
 +5V_HDMI_CON: disabling
 ALSA device list:
   No soundcards found.
-Freeing unused kernel memory: 1088K
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
+Freeing unused kernel memory: 1152K
 
 System initialization...
 
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-09-24 17:13:49.894381363 -0700
+++ /tmp/foob	2017-09-24 17:13:49.894381363 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 755866 us
-Basic modules init took 348743 us
+Tegraboot started after 883229 us
+Basic modules init took 348776 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14370 us
-Carveout took 252696 us
-CPU initialization took 92435 us
-Total time taken by TegraBoot 708244 us
+Next stage binary read took 14385 us
+Carveout took 252675 us
+CPU initialization took 92464 us
+Total time taken by TegraBoot 708300 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.594] LPDDR4 Training: Number of tables = 10
+64b[0001.719] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16439808 bytes read in 835 ms (18.8 MiB/s)
+16505344 bytes read in 829 ms (19 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-45170 bytes read in 455 ms (96.7 KiB/s)
+45170 bytes read in 453 ms (96.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-199401-g2bd6bf0 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sat Sep 16 16:42:19 PDT 2017
+Linux version 4.14.0-rc2-199764-ge19b205 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Sep 24 17:10:07 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
@@ -173,15 +173,15 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2982628K/3131392K available (9212K kernel code, 1142K rwdata, 4568K rodata, 1088K init, 402K bss, 132380K reserved, 16384K cma-reserved)
+Memory: 2982564K/3131392K available (9212K kernel code, 1142K rwdata, 4572K rodata, 1152K init, 402K bss, 132444K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
     .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
-      .init : 0xffff000008e00000 - 0xffff000008f10000   (  1088 KB)
-      .data : 0xffff000008f10000 - 0xffff00000902da00   (  1143 KB)
-       .bss : 0xffff00000902da00 - 0xffff000009092470   (   403 KB)
+      .init : 0xffff000008e00000 - 0xffff000008f20000   (  1152 KB)
+      .data : 0xffff000008f20000 - 0xffff00000903da00   (  1143 KB)
+       .bss : 0xffff00000903da00 - 0xffff0000090a2470   (   403 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -233,7 +233,7 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f15000)
+vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f25000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -352,7 +352,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1088K
+Freeing unused kernel memory: 1152K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-09-24 17:13:49.898381379 -0700
+++ /tmp/foob	2017-09-24 17:13:49.902381396 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-199401-g2bd6bf0 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sat Sep 16 16:42:19 PDT 2017
+Linux version 4.14.0-rc2-199764-ge19b205 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Sep 24 17:10:07 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
@@ -31,15 +31,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837892K/2988032K available (9212K kernel code, 1142K rwdata, 4568K rodata, 1088K init, 402K bss, 133756K reserved, 16384K cma-reserved)
+Memory: 2837828K/2988032K available (9212K kernel code, 1142K rwdata, 4572K rodata, 1152K init, 402K bss, 133820K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
     .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
-      .init : 0xffff000008e00000 - 0xffff000008f10000   (  1088 KB)
-      .data : 0xffff000008f10000 - 0xffff00000902da00   (  1143 KB)
-       .bss : 0xffff00000902da00 - 0xffff000009092470   (   403 KB)
+      .init : 0xffff000008e00000 - 0xffff000008f20000   (  1152 KB)
+      .data : 0xffff000008f20000 - 0xffff00000903da00   (  1143 KB)
+       .bss : 0xffff00000903da00 - 0xffff0000090a2470   (   403 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -108,7 +108,7 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f15000)
+vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f25000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -258,9 +258,8 @@
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
+max77686-rtc max77620-rtc: setting system clock to 2001-07-06 05:34:13 UTC (994397653)
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2001-06-28 05:06:29 UTC (993704789)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -268,11 +267,12 @@
 PP_LDO6: disabling
 PPLCD_VDD: disabling
 PP1800_LCDIO: disabling
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
-Freeing unused kernel memory: 1088K
+Freeing unused kernel memory: 1152K
 
 System initialization...
 
