Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-05-24 18:38:13.258169658 -0700
+++ /tmp/foob	2015-05-24 18:38:13.258169658 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.1.0-rc4+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon May 18 10:35:17 PDT 2015
+Linux version 4.1.0-rc5+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun May 24 18:34:25 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -18,7 +18,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 939548K/1048576K available (5385K kernel code, 435K rwdata, 2136K rodata, 324K init, 202K bss, 92644K reserved, 16384K cma-reserved)
+Memory: 939416K/1048576K available (5385K kernel code, 437K rwdata, 2168K rodata, 420K init, 204K bss, 92776K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -27,9 +27,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc0007db000 - 0xffffffc00082c000   (   324 KB)
-      .text : 0xffffffc000080000 - 0xffffffc0007da564   (  7530 KB)
-      .data : 0xffffffc000832000 - 0xffffffc00089ec00   (   435 KB)
+      .init : 0xffffffc0007e3000 - 0xffffffc00084c000   (   420 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0007e2564   (  7562 KB)
+      .data : 0xffffffc000852000 - 0xffffffc0008bf400   (   437 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -60,7 +60,7 @@
 NET: Registered protocol family 16
 cpuidle: using governor ladder
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffffc000839000, 1 data @ ffffffc000838000)
+vdso: 2 pages (1 code @ ffffffc000859000, 1 data @ ffffffc000858000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -129,8 +129,8 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 324K (ffffffc0007db000 - ffffffc00082c000)
-Freeing alternatives memory: 12K (ffffffc00082c000 - ffffffc00082f000)
+Freeing unused kernel memory: 420K (ffffffc0007e3000 - ffffffc00084c000)
+Freeing alternatives memory: 12K (ffffffc00084c000 - ffffffc00084f000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-05-24 18:38:13.266169653 -0700
+++ /tmp/foob	2015-05-24 18:38:13.270169650 -0700
@@ -37,10 +37,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
-	 3.4 MiB/s
+	 #####################################
+	 3.3 MiB/s
 done
-Bytes transferred = 6239792 (5f3630 hex)
+Bytes transferred = 6260232 (5f8608 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5f3630 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5f8608 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-rc4-12997-ge260818 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon May 18 10:33:59 PDT 2015
+Linux version 4.1.0-rc5-13213-gba155e2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 24 18:33:23 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f25e40, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f38680, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999244K/2096128K available (7987K kernel code, 1023K rwdata, 3588K rodata, 836K init, 318K bss, 31348K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999168K/2096128K available (7991K kernel code, 1025K rwdata, 3612K rodata, 880K init, 318K bss, 31424K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d56f50   (11580 kB)
-      .init : 0xc0d57000 - 0xc0e28000   ( 836 kB)
-      .data : 0xc0e28000 - 0xc0f27e20   (1024 kB)
-       .bss : 0xc0f2a000 - 0xc0f79a68   ( 319 kB)
+      .text : 0xc0208000 - 0xc0d5df48   (11608 kB)
+      .init : 0xc0d5e000 - 0xc0e3a000   ( 880 kB)
+      .data : 0xc0e3a000 - 0xc0f3a660   (1026 kB)
+       .bss : 0xc0f3d000 - 0xc0f8ca68   ( 319 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -302,7 +302,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-23 12:59:16 UTC (972305956)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-29 20:58:53 UTC (972853133)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -313,7 +313,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 836K (c0d57000 - c0e28000)
+Freeing unused kernel memory: 880K (c0d5e000 - c0e3a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.603726] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.618722] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-05-24 18:38:13.274169648 -0700
+++ /tmp/foob	2015-05-24 18:38:13.274169648 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -39,18 +39,18 @@
 	 ########
 	 3.3 MiB/s
 done
-Bytes transferred = 4884248 (4a8718 hex)
+Bytes transferred = 4884464 (4a87f0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4a8718 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4a87f0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-rc4-12997-ge260818 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon May 18 10:34:16 PDT 2015
+Linux version 4.1.0-rc5-13213-gba155e2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 24 18:33:34 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2003056K/2096128K available (6696K kernel code, 312K rwdata, 2260K rodata, 312K init, 351K bss, 27536K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2003056K/2096128K available (6700K kernel code, 312K rwdata, 2260K rodata, 308K init, 351K bss, 27536K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,8 +90,8 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc08c74e4   (8958 kB)
-      .init : 0xc08c8000 - 0xc0916000   ( 312 kB)
+      .text : 0xc0008000 - 0xc08c84ec   (8962 kB)
+      .init : 0xc08c9000 - 0xc0916000   ( 308 kB)
       .data : 0xc0916000 - 0xc0964020   ( 313 kB)
        .bss : 0xc0967000 - 0xc09bee14   ( 352 kB)
 Preemptible hierarchical RCU implementation.
@@ -293,7 +293,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-23 12:59:44 UTC (972305984)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-29 20:59:22 UTC (972853162)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -305,7 +305,7 @@
 EXT3-fs (mmcblk0p1): mounted filesystem with writeback data mode
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 312K (c08c8000 - c0916000)
+Freeing unused kernel memory: 308K (c08c9000 - c0916000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.789708] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.789703] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-05-24 18:38:13.282169644 -0700
+++ /tmp/foob	2015-05-24 18:38:13.282169644 -0700
@@ -39,21 +39,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
+	 #####################################
 	 3.2 MiB/s
 done
-Bytes transferred = 6239792 (5f3630 hex)
+Bytes transferred = 6260232 (5f8608 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.9 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 63003 (f61b hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5f3630 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5f8608 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 8201261a
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-rc4-12997-ge260818 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon May 18 10:33:59 PDT 2015
+Linux version 4.1.0-rc5-13213-gba155e2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 24 18:33:23 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f25e40, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f38680, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999140K/2096128K available (7987K kernel code, 1023K rwdata, 3588K rodata, 836K init, 318K bss, 31452K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999064K/2096128K available (7991K kernel code, 1025K rwdata, 3612K rodata, 880K init, 318K bss, 31528K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d56f50   (11580 kB)
-      .init : 0xc0d57000 - 0xc0e28000   ( 836 kB)
-      .data : 0xc0e28000 - 0xc0f27e20   (1024 kB)
-       .bss : 0xc0f2a000 - 0xc0f79a68   ( 319 kB)
+      .text : 0xc0208000 - 0xc0d5df48   (11608 kB)
+      .init : 0xc0d5e000 - 0xc0e3a000   ( 880 kB)
+      .data : 0xc0e3a000 - 0xc0f3a660   (1026 kB)
+       .bss : 0xc0f3d000 - 0xc0f8ca68   ( 319 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -371,7 +371,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 836K (c0d57000 - c0e28000)
+Freeing unused kernel memory: 880K (c0d5e000 - c0e3a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-05-24 18:38:13.290169638 -0700
+++ /tmp/foob	2015-05-24 18:38:13.290169638 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ########
-	 3.2 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 4884248 (4a8718 hex)
+Bytes transferred = 4884464 (4a87f0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.7 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 63003 (f61b hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4a8718 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4a87f0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 8201261a
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-rc4-12997-ge260818 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon May 18 10:34:16 PDT 2015
+Linux version 4.1.0-rc5-13213-gba155e2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 24 18:33:34 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002952K/2096128K available (6696K kernel code, 312K rwdata, 2260K rodata, 312K init, 351K bss, 27640K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002952K/2096128K available (6700K kernel code, 312K rwdata, 2260K rodata, 308K init, 351K bss, 27640K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,8 +92,8 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc08c74e4   (8958 kB)
-      .init : 0xc08c8000 - 0xc0916000   ( 312 kB)
+      .text : 0xc0008000 - 0xc08c84ec   (8962 kB)
+      .init : 0xc08c9000 - 0xc0916000   ( 308 kB)
       .data : 0xc0916000 - 0xc0964020   ( 313 kB)
        .bss : 0xc0967000 - 0xc09bee14   ( 352 kB)
 Preemptible hierarchical RCU implementation.
@@ -344,7 +344,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:52 UTC (949363252)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:51 UTC (949363251)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 +2.8V_RUN_CAM_AF: disabling
@@ -363,7 +363,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 312K (c08c8000 - c0916000)
+Freeing unused kernel memory: 308K (c08c9000 - c0916000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-05-24 18:38:13.294169636 -0700
+++ /tmp/foob	2015-05-24 18:38:13.298169634 -0700
@@ -43,21 +43,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
-	 4 MiB/s
+	 #####################################
+	 3.9 MiB/s
 done
-Bytes transferred = 6239792 (5f3630 hex)
+Bytes transferred = 6260232 (5f8608 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5f3630 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5f8608 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -65,7 +65,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-rc4-12997-ge260818 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon May 18 10:33:59 PDT 2015
+Linux version 4.1.0-rc5-13213-gba155e2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 24 18:33:23 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -73,7 +73,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0f25e40, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0f38680, node_mem_map eeff8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -86,7 +86,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 960052K/1048576K available (7987K kernel code, 1023K rwdata, 3588K rodata, 836K init, 318K bss, 22988K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 959976K/1048576K available (7991K kernel code, 1025K rwdata, 3612K rodata, 880K init, 318K bss, 23064K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,10 +94,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d56f50   (11580 kB)
-      .init : 0xc0d57000 - 0xc0e28000   ( 836 kB)
-      .data : 0xc0e28000 - 0xc0f27e20   (1024 kB)
-       .bss : 0xc0f2a000 - 0xc0f79a68   ( 319 kB)
+      .text : 0xc0208000 - 0xc0d5df48   (11608 kB)
+      .init : 0xc0d5e000 - 0xc0e3a000   ( 880 kB)
+      .data : 0xc0e3a000 - 0xc0f3a660   (1026 kB)
+       .bss : 0xc0f3d000 - 0xc0f8ca68   ( 319 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -345,10 +345,10 @@
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-hub 1-1:1.0: USB hub found
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-rtc-em3027 2-0056: setting system clock to 2014-12-25 20:22:24 UTC (1419538944)
+rtc-em3027 2-0056: setting system clock to 2014-12-25 20:33:21 UTC (1419539601)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
@@ -359,7 +359,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 836K (c0d57000 - c0e28000)
+Freeing unused kernel memory: 880K (c0d5e000 - c0e3a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-05-24 18:38:13.302169632 -0700
+++ /tmp/foob	2015-05-24 18:38:13.306169629 -0700
@@ -45,18 +45,18 @@
 	 ########
 	 4.1 MiB/s
 done
-Bytes transferred = 4884248 (4a8718 hex)
+Bytes transferred = 4884464 (4a87f0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2.9 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4a8718 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4a87f0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-rc4-12997-ge260818 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon May 18 10:34:16 PDT 2015
+Linux version 4.1.0-rc5-13213-gba155e2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 24 18:33:34 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963864K/1048576K available (6696K kernel code, 312K rwdata, 2260K rodata, 312K init, 351K bss, 19176K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 963864K/1048576K available (6700K kernel code, 312K rwdata, 2260K rodata, 308K init, 351K bss, 19176K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,8 +97,8 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc08c74e4   (8958 kB)
-      .init : 0xc08c8000 - 0xc0916000   ( 312 kB)
+      .text : 0xc0008000 - 0xc08c84ec   (8962 kB)
+      .init : 0xc08c9000 - 0xc0916000   ( 308 kB)
       .data : 0xc0916000 - 0xc0964020   ( 313 kB)
        .bss : 0xc0967000 - 0xc09bee14   ( 352 kB)
 Preemptible hierarchical RCU implementation.
@@ -341,7 +341,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-25 20:22:44 UTC (1419538964)
+rtc-em3027 2-0056: setting system clock to 2014-12-25 20:33:39 UTC (1419539619)
 ALSA device list:
   #0: tegra-trimslice
 hub 1-1:1.0: USB hub found
@@ -354,7 +354,7 @@
 EXT3-fs (mmcblk0p1): mounted filesystem with writeback data mode
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 312K (c08c8000 - c0916000)
+Freeing unused kernel memory: 308K (c08c9000 - c0916000)
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-05-24 18:38:13.310169626 -0700
+++ /tmp/foob	2015-05-24 18:38:13.314169624 -0700
@@ -37,21 +37,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
-	 3.4 MiB/s
+	 #####################################
+	 3.3 MiB/s
 done
-Bytes transferred = 6239792 (5f3630 hex)
+Bytes transferred = 6260232 (5f8608 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.5 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51191 (c7f7 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5f3630 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5f8608 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff7f6 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-rc4-12997-ge260818 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon May 18 10:33:59 PDT 2015
+Linux version 4.1.0-rc5-13213-gba155e2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 24 18:33:23 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f25e40, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f38680, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999164K/2096128K available (7987K kernel code, 1023K rwdata, 3588K rodata, 836K init, 318K bss, 31428K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999088K/2096128K available (7991K kernel code, 1025K rwdata, 3612K rodata, 880K init, 318K bss, 31504K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d56f50   (11580 kB)
-      .init : 0xc0d57000 - 0xc0e28000   ( 836 kB)
-      .data : 0xc0e28000 - 0xc0f27e20   (1024 kB)
-       .bss : 0xc0f2a000 - 0xc0f79a68   ( 319 kB)
+      .text : 0xc0208000 - 0xc0d5df48   (11608 kB)
+      .init : 0xc0d5e000 - 0xc0e3a000   ( 880 kB)
+      .data : 0xc0e3a000 - 0xc0f3a660   (1026 kB)
+       .bss : 0xc0f3d000 - 0xc0f8ca68   ( 319 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -374,7 +374,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 836K (c0d57000 - c0e28000)
+Freeing unused kernel memory: 880K (c0d5e000 - c0e3a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.208476] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.213452] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-05-24 18:38:13.318169622 -0700
+++ /tmp/foob	2015-05-24 18:38:13.322169620 -0700
@@ -39,18 +39,18 @@
 	 ########
 	 3.4 MiB/s
 done
-Bytes transferred = 4884248 (4a8718 hex)
+Bytes transferred = 4884464 (4a87f0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51191 (c7f7 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4a8718 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4a87f0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff7f6 ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-rc4-12997-ge260818 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon May 18 10:34:16 PDT 2015
+Linux version 4.1.0-rc5-13213-gba155e2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 24 18:33:34 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002976K/2096128K available (6696K kernel code, 312K rwdata, 2260K rodata, 312K init, 351K bss, 27616K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002976K/2096128K available (6700K kernel code, 312K rwdata, 2260K rodata, 308K init, 351K bss, 27616K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,8 +90,8 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc08c74e4   (8958 kB)
-      .init : 0xc08c8000 - 0xc0916000   ( 312 kB)
+      .text : 0xc0008000 - 0xc08c84ec   (8962 kB)
+      .init : 0xc08c9000 - 0xc0916000   ( 308 kB)
       .data : 0xc0916000 - 0xc0964020   ( 313 kB)
        .bss : 0xc0967000 - 0xc09bee14   ( 352 kB)
 Preemptible hierarchical RCU implementation.
@@ -367,7 +367,7 @@
 EXT3-fs (mmcblk0p1): mounted filesystem with writeback data mode
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 312K (c08c8000 - c0916000)
+Freeing unused kernel memory: 308K (c08c9000 - c0916000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.466213] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.466224] usb 2-1: new high-speed USB device number 2 using tegra-ehci
