Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-01-04 04:51:37.246902793 -0800
+++ /tmp/foob	2016-01-04 04:51:37.246902793 -0800
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.4.0-rc7+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Dec 27 18:34:39 PST 2015
+Linux version 4.4.0-rc8+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jan 4 04:47:06 PST 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -89,7 +89,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.632:1): initialized
+audit: type=2000 audit(0.548:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-04 04:51:37.254902785 -0800
+++ /tmp/foob	2016-01-04 04:51:37.258902782 -0800
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ########################################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6543816 (63d9c8 hex)
+Bytes transferred = 6544120 (63daf8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x63d9c8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x63daf8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc7-55671-g74bf8ef (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Dec 27 18:33:14 PST 2015
+Linux version 4.4.0-rc8-55753-g1683098 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jan 4 04:45:41 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fc23c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fc2400, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -88,9 +88,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dbacc8   (11980 kB)
+      .text : 0xc0208000 - 0xc0dbacd4   (11980 kB)
       .init : 0xc0dbb000 - 0xc0ec0000   (1044 kB)
-      .data : 0xc0ec0000 - 0xc0fc4320   (1041 kB)
+      .data : 0xc0ec0000 - 0xc0fc4360   (1041 kB)
        .bss : 0xc0fc7000 - 0xc101c3e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -297,7 +297,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-09 11:33:04 UTC (963142384)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-16 21:45:56 UTC (963783956)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -309,4 +309,4 @@
 Freeing unused kernel memory: 1044K (c0dbb000 - c0ec0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.234024] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.239022] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-04 04:51:37.262902778 -0800
+++ /tmp/foob	2016-01-04 04:51:37.266902775 -0800
@@ -39,18 +39,18 @@
 	 ##############################
 	 3.3 MiB/s
 done
-Bytes transferred = 5204112 (4f6890 hex)
+Bytes transferred = 5204160 (4f68c0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f6890 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f68c0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc7-55671-g74bf8ef (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Dec 27 18:33:31 PST 2015
+Linux version 4.4.0-rc8-55753-g1683098 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jan 4 04:45:59 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -70,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a0e740, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a0e780, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -91,9 +91,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09640b4   (9585 kB)
+      .text : 0xc0008000 - 0xc09640b8   (9585 kB)
       .init : 0xc0965000 - 0xc09b8000   ( 332 kB)
-      .data : 0xc09b8000 - 0xc0a10260   ( 353 kB)
+      .data : 0xc09b8000 - 0xc0a102a0   ( 353 kB)
        .bss : 0xc0a13000 - 0xc0a6e090   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -300,7 +300,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-09 11:33:30 UTC (963142410)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-16 21:46:29 UTC (963783989)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -314,4 +314,4 @@
 Freeing unused kernel memory: 332K (c0965000 - c09b8000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.545583] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.565561] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-04 04:51:37.270902772 -0800
+++ /tmp/foob	2016-01-04 04:51:37.274902767 -0800
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ########################################################
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6543816 (63d9c8 hex)
+Bytes transferred = 6544120 (63daf8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x63d9c8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x63daf8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc7-55671-g74bf8ef (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Dec 27 18:33:14 PST 2015
+Linux version 4.4.0-rc8-55753-g1683098 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jan 4 04:45:41 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -70,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fc23c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fc2400, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -91,9 +91,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dbacc8   (11980 kB)
+      .text : 0xc0208000 - 0xc0dbacd4   (11980 kB)
       .init : 0xc0dbb000 - 0xc0ec0000   (1044 kB)
-      .data : 0xc0ec0000 - 0xc0fc4320   (1041 kB)
+      .data : 0xc0ec0000 - 0xc0fc4360   (1041 kB)
        .bss : 0xc0fc7000 - 0xc101c3e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -287,17 +287,17 @@
 +3.3V_SYS: supplied by +VDD_MUX
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.35V_LP0(sd2): supplied by +5V_SYS
-+1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
 mmc0: MAN_BKOPS_EN bit is not set
 mmc0: new high speed MMC card at address 0001
-tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
-tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 mmcblk0: mmc0:0001 SEM16G 14.6 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
++1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
+tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
+tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
@@ -364,8 +364,8 @@
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 VFS: Mounted root (ext4 filesystem) on device 179:1.
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
 Freeing unused kernel memory: 1044K (c0dbb000 - c0ec0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-04 04:51:37.278902764 -0800
+++ /tmp/foob	2016-01-04 04:51:37.278902764 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5204112 (4f6890 hex)
+Bytes transferred = 5204160 (4f68c0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.9 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x4f6890 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x4f68c0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc7-55671-g74bf8ef (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Dec 27 18:33:31 PST 2015
+Linux version 4.4.0-rc8-55753-g1683098 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jan 4 04:45:59 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -72,7 +72,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a0e740, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a0e780, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -93,9 +93,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09640b4   (9585 kB)
+      .text : 0xc0008000 - 0xc09640b8   (9585 kB)
       .init : 0xc0965000 - 0xc09b8000   ( 332 kB)
-      .data : 0xc09b8000 - 0xc0a10260   ( 353 kB)
+      .data : 0xc09b8000 - 0xc0a102a0   ( 353 kB)
        .bss : 0xc0a13000 - 0xc0a6e090   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -307,16 +307,42 @@
 tegra-pcie 1003000.pcie-controller: link 0 down, ignoring
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
-tegra-pcie 1003000.pcie-controller: link 1 down, retrying
-tegra-pcie 1003000.pcie-controller: link 1 down, retrying
-tegra-pcie 1003000.pcie-controller: link 1 down, retrying
-tegra-pcie 1003000.pcie-controller: link 1 down, ignoring
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
 pci_bus 0000:00: root bus resource [mem 0x13000000-0x1fffffff]
 pci_bus 0000:00: root bus resource [mem 0x20000000-0x3fffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci_bus 0000:00: root bus resource [io  0x1000-0xffff]
-PCI: bus0: Fast back to back transfers enabled
+pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
+pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
+PCI: bus0: Fast back to back transfers disabled
+pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
+pci 0000:01:00.0: [10ec:8168] type 00 class 0x020000
+pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
+pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
+pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
+pci 0000:01:00.0: supports D1 D2
+pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
+PCI: bus1: Fast back to back transfers disabled
+pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
+pci 0000:00:02.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
+pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
+pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
+pci 0000:01:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
+pci 0000:01:00.0: BAR 2: assigned [mem 0x13000000-0x13000fff 64bit]
+pci 0000:01:00.0: BAR 0: assigned [io  0x1000-0x10ff]
+pci 0000:00:02.0: PCI bridge to [bus 01]
+pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
+pci 0000:00:02.0:   bridge window [mem 0x13000000-0x130fffff]
+pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
+pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
+pcieport 0000:00:02.0: enabling device (0140 -> 0143)
+pcieport 0000:00:02.0: Signaling PME through PCIe PME interrupt
+pci 0000:01:00.0: Signaling PME through PCIe PME interrupt
+pcie_pme 0000:00:02.0:pcie01: service driver pcie_pme loaded
+r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
+r8169 0000:01:00.0: enabling device (0140 -> 0143)
+r8169 0000:01:00.0 eth0: RTL8168g/8111g at 0xf08f8000, 00:04:4b:25:b3:b4, XID 0c000800 IRQ 388
+r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +5V_HDMI_CON: supplied by +5V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 +3.3V_RUN: supplied by +3.3V_SYS
@@ -353,7 +379,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:53 UTC (946684853)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:52 UTC (946684852)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -367,10 +393,10 @@
   #1: NVIDIA Tegra Jetson TK1
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
 Freeing unused kernel memory: 332K (c0965000 - c09b8000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-04 04:51:37.286902757 -0800
+++ /tmp/foob	2016-01-04 04:51:37.286902757 -0800
@@ -48,7 +48,7 @@
 	 ########################################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6543816 (63d9c8 hex)
+Bytes transferred = 6544120 (63daf8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x63d9c8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x63daf8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc7-55671-g74bf8ef (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Dec 27 18:33:14 PST 2015
+Linux version 4.4.0-rc8-55753-g1683098 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jan 4 04:45:41 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0fc23c0, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c0fc2400, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -96,9 +96,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dbacc8   (11980 kB)
+      .text : 0xc0208000 - 0xc0dbacd4   (11980 kB)
       .init : 0xc0dbb000 - 0xc0ec0000   (1044 kB)
-      .data : 0xc0ec0000 - 0xc0fc4320   (1041 kB)
+      .data : 0xc0ec0000 - 0xc0fc4360   (1041 kB)
        .bss : 0xc0fc7000 - 0xc101c3e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -333,7 +333,7 @@
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-27 03:19:21 UTC (1419650361)
+rtc-em3027 2-0056: setting system clock to 2014-12-27 04:47:45 UTC (1419655665)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
@@ -346,4 +346,4 @@
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.270742] asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
+root@(none):/# [    3.270825] asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-04 04:51:37.294902749 -0800
+++ /tmp/foob	2016-01-04 04:51:37.294902749 -0800
@@ -47,7 +47,7 @@
 	 ##############################
 	 4.1 MiB/s
 done
-Bytes transferred = 5204112 (4f6890 hex)
+Bytes transferred = 5204160 (4f68c0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4f6890 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4f68c0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc7-55671-g74bf8ef (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Dec 27 18:33:31 PST 2015
+Linux version 4.4.0-rc8-55753-g1683098 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jan 4 04:45:59 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0a0e740, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0a0e780, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -99,9 +99,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09640b4   (9585 kB)
+      .text : 0xc0008000 - 0xc09640b8   (9585 kB)
       .init : 0xc0965000 - 0xc09b8000   ( 332 kB)
-      .data : 0xc09b8000 - 0xc0a10260   ( 353 kB)
+      .data : 0xc09b8000 - 0xc0a102a0   ( 353 kB)
        .bss : 0xc0a13000 - 0xc0a6e090   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -348,7 +348,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-27 03:19:43 UTC (1419650383)
+rtc-em3027 2-0056: setting system clock to 2014-12-27 04:48:04 UTC (1419655684)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-04 04:51:37.298902745 -0800
+++ /tmp/foob	2016-01-04 04:51:37.302902742 -0800
@@ -40,18 +40,18 @@
 	 ########################################################
 	 3.4 MiB/s
 done
-Bytes transferred = 6543816 (63d9c8 hex)
+Bytes transferred = 6544120 (63daf8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x63d9c8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x63daf8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc7-55671-g74bf8ef (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Dec 27 18:33:14 PST 2015
+Linux version 4.4.0-rc8-55753-g1683098 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jan 4 04:45:41 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fc23c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fc2400, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -88,9 +88,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dbacc8   (11980 kB)
+      .text : 0xc0208000 - 0xc0dbacd4   (11980 kB)
       .init : 0xc0dbb000 - 0xc0ec0000   (1044 kB)
-      .data : 0xc0ec0000 - 0xc0fc4320   (1041 kB)
+      .data : 0xc0ec0000 - 0xc0fc4360   (1041 kB)
        .bss : 0xc0fc7000 - 0xc101c3e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -279,16 +279,16 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
+tegra-pcie 3000.pcie-controller: 2x3 configuration
+sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
+sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
+tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
-tegra-pcie 3000.pcie-controller: 2x3 configuration
-sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
-sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
-tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
  mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -364,4 +364,4 @@
 Freeing unused kernel memory: 1044K (c0dbb000 - c0ec0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.728790] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.723431] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-04 04:51:37.306902738 -0800
+++ /tmp/foob	2016-01-04 04:51:37.310902735 -0800
@@ -27,7 +27,6 @@
 Tegra30 (Beaver) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra30 (Beaver) # setenv ipaddr 192.168.100.101; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra30 (Beaver) # tftpboot 0x81000000 zImage.tegra_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'zImage.tegra_defconfig'.
@@ -38,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5204112 (4f6890 hex)
+Bytes transferred = 5204160 (4f68c0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f6890 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f68c0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -62,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc7-55671-g74bf8ef (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Dec 27 18:33:31 PST 2015
+Linux version 4.4.0-rc8-55753-g1683098 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jan 4 04:45:59 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -71,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a0e740, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a0e780, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -92,9 +91,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09640b4   (9585 kB)
+      .text : 0xc0008000 - 0xc09640b8   (9585 kB)
       .init : 0xc0965000 - 0xc09b8000   ( 332 kB)
-      .data : 0xc09b8000 - 0xc0a10260   ( 353 kB)
+      .data : 0xc09b8000 - 0xc0a102a0   ( 353 kB)
        .bss : 0xc0a13000 - 0xc0a6e090   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -371,4 +370,4 @@
 Freeing unused kernel memory: 332K (c0965000 - c09b8000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.170553] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.170579] usb 2-1: new high-speed USB device number 2 using tegra-ehci
