Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:06.914438468 -0800
+++ /tmp/foob	2016-11-21 03:44:06.918438484 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc5-152502-ga25f094 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Nov 14 02:07:04 PST 2016
+Linux version 4.9.0-rc6-152872-g9c76358 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Nov 21 03:37:03 PST 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -100,7 +100,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.660:1): initialized
+audit: type=2000 audit(0.652:1): initialized
 workingset: timestamp_bits=46 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -167,7 +167,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-11-14 10:07:41 UTC (1479118061)
+rtc-pl031 9010000.pl031: setting system clock to 2016-11-21 11:37:39 UTC (1479728259)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:06.950438611 -0800
+++ /tmp/foob	2016-11-21 03:44:06.950438611 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 7286552 (6f2f18 hex)
+Bytes transferred = 7286920 (6f3088 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6f2f18 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6f3088 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc5-128069-ga25f094 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 14 02:04:24 PST 2016
+Linux version 4.9.0-rc6-128439-g9c76358 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 21 03:34:34 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1427f80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1427fc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -94,7 +94,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1429a60   (1191 kB)
+      .data : 0xc1300000 - 0xc1429aa0   (1191 kB)
        .bss : 0xc142b000 - 0xc1480fc8   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -303,20 +303,20 @@
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 usb1_vbus: supplied by vdd-sys-5v0
+mmc0: new high speed SDHC card at address aaaa
 usb2_vbus: supplied by vdd-sys-5v0
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
  mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 Console: switching to colour frame buffer device 240x75
+ mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 tegra-ehci 7d008000.usb: EHCI Host Controller
@@ -326,7 +326,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-09-07 21:34:33 UTC (968362473)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-09-14 23:02:32 UTC (968972552)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -338,4 +338,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.435224] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.425233] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:06.970438691 -0800
+++ /tmp/foob	2016-11-21 03:44:06.974438707 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################################
-	 3.3 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 5453296 (5335f0 hex)
+Bytes transferred = 5454800 (533bd0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5335f0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x533bd0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc5-128069-ga25f094 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 14 02:04:10 PST 2016
+Linux version 4.9.0-rc6-128439-g9c76358 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 21 03:34:33 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -132,7 +132,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.602985] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.602779] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 3 registered
 dvdd_ts: Failed to request enable GPIO61: -517
@@ -319,7 +319,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-09-07 21:31:51 UTC (968362311)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-09-14 23:05:31 UTC (968972731)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -333,5 +333,5 @@
 Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.931570] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.921250] usb 1-1: new high-speed USB device number 2 using tegra-ehci
 asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:06.990438770 -0800
+++ /tmp/foob	2016-11-21 03:44:06.994438786 -0800
@@ -34,20 +34,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 3.7 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 7286552 (6f2f18 hex)
+Bytes transferred = 7286920 (6f3088 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6f2f18 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6f3088 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc5-128069-ga25f094 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 14 02:04:24 PST 2016
+Linux version 4.9.0-rc6-128439-g9c76358 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 21 03:34:34 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -66,7 +66,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1427f80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1427fc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1429a60   (1191 kB)
+      .data : 0xc1300000 - 0xc1429aa0   (1191 kB)
        .bss : 0xc142b000 - 0xc1480fc8   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -309,8 +309,8 @@
 +3.3V_LP0: supplied by +3.3V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 +5V_SATA: supplied by +5V_SYS
-mmc0: new high speed MMC card at address 0001
 +12V_SATA: supplied by +VDD_MUX
+mmc0: new high speed MMC card at address 0001
 usb2-0 supply vbus not found, using dummy regulator
 usb2-1 supply vbus not found, using dummy regulator
 usb3-0 supply vbus not found, using dummy regulator
@@ -417,23 +417,23 @@
 [drm] No driver support for vblank timestamp query.
 drm drm: No connectors reported connected with modes
 [drm] Cannot find any crtc or sizes - going 1024x768
-ata1: SATA link down (SStatus 0 SControl 300)
 mmc1: new high speed SDHC card at address aaaa
 mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
+ata1: SATA link down (SStatus 0 SControl 300)
  mmcblk1: p1 p2 p3
+Console: switching to colour frame buffer device 128x48
+drm drm: fb0:  frame buffer device
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
-Console: switching to colour frame buffer device 128x48
  mmcblk0: p1
-drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:12 UTC (946684812)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:07.010438850 -0800
+++ /tmp/foob	2016-11-21 03:44:07.014438866 -0800
@@ -32,9 +32,9 @@
 	 #################################################################
 	 #################################################################
 	 ###############################################
-	 4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5453296 (5335f0 hex)
+Bytes transferred = 5454800 (533bd0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -45,7 +45,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5335f0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x533bd0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc5-128069-ga25f094 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 14 02:04:10 PST 2016
+Linux version 4.9.0-rc6-128439-g9c76358 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 21 03:34:33 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -127,7 +127,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.662006] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.661628] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 +5V_SYS: supplied by +VDD_MUX
 +3.3V_SYS: supplied by +VDD_MUX
@@ -400,9 +400,9 @@
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
@@ -417,11 +417,11 @@
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
- mmcblk1: p1 p2 p3
 pci 0000:00:01.0: nv_msi_ht_cap_quirk didn't locate host bridge
 pcieport 0000:00:01.0: enabling device (0140 -> 0142)
 pcieport 0000:00:01.0: Signaling PME through PCIe PME interrupt
 pci 0000:01:00.0: Signaling PME through PCIe PME interrupt
+ mmcblk1: p1 p2 p3
 pcie_pme 0000:00:01.0:pcie001: service driver pcie_pme loaded
 pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
 pcieport 0000:00:02.0: enabling device (0140 -> 0143)
@@ -442,7 +442,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:53 UTC (946684913)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:07.018438882 -0800
+++ /tmp/foob	2016-11-21 03:44:07.022438897 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -42,20 +42,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 3.5 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 7286552 (6f2f18 hex)
+Bytes transferred = 7286920 (6f3088 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.6 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 67899 (1093b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6f2f18 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6f3088 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301393a
@@ -63,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc5-128069-ga25f094 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 14 02:04:24 PST 2016
+Linux version 4.9.0-rc6-128439-g9c76358 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 21 03:34:34 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c1427f80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1427fc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -98,7 +98,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1429a60   (1191 kB)
+      .data : 0xc1300000 - 0xc1429aa0   (1191 kB)
        .bss : 0xc142b000 - 0xc1480fc8   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:07.034438945 -0800
+++ /tmp/foob	2016-11-21 03:44:07.034438945 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -40,9 +40,9 @@
 	 #################################################################
 	 #################################################################
 	 ###############################################
-	 3.5 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 5453296 (5335f0 hex)
+Bytes transferred = 5454800 (533bd0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -53,7 +53,7 @@
 done
 Bytes transferred = 67899 (1093b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5335f0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x533bd0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301393a
@@ -62,7 +62,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc5-128069-ga25f094 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 14 02:04:10 PST 2016
+Linux version 4.9.0-rc6-128439-g9c76358 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 21 03:34:33 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -136,7 +136,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.576511] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.576587] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 0 registered
 +5V_SYS: supplied by +VDD_MUX
@@ -284,13 +284,13 @@
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
 max98090 0-0010: MAX98090 REVID=0x43
 max98090 0-0010: use default 2.8v micbias
+tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
 input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
 input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 NET: Registered protocol family 10
@@ -353,8 +353,8 @@
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 usb 1-2: new high-speed USB device number 2 using tegra-xusb
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
 GPT:Primary header alternate_lba != Alt. header my_lba
@@ -379,13 +379,13 @@
   #0: tegra-hda at 0x70038000 irq 341
   #1: GoogleNyanBig
 EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
+uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
+input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
-uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
-input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.043052] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+root@(none):/# [    3.033116] usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:07.058439041 -0800
+++ /tmp/foob	2016-11-21 03:44:07.058439041 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801450 us
+Bootloader-Cpu Init at (time stamp): 1800566 us
 
 Pinmux changes applied in kernel way
 
@@ -59,23 +59,23 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 4 us
+Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801423  us
-Bootloader (AosInit) time: 733468 us
-Bootloader Main Init time: 180074 us
-BlockDevMgrInit time: 157535 us
+Miscellaneous avp time: 1800540  us
+Bootloader (AosInit) time: 733490 us
+Bootloader Main Init time: 180208 us
+BlockDevMgrInit time: 157551 us
 PartMgrInit time: 8 us
 StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 77023 us 
-Display init time: 574437 us
-Kernel read time: 377 msec
+Partition Table load: 77020 us 
+Display init time: 574436 us
+Kernel read time: 378 msec
 Load Kernel Size:   16304128 Bytes
-Bl Kernel ReadSpeed: 43247000 Bytes/sec
-Pre-Kernel Jump time: 1741186 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5845270 us
+Bl Kernel ReadSpeed: 43132000 Bytes/sec
+Pre-Kernel Jump time: 1740329 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5844413 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.127:1): initialized
+audit: type=2000 audit(1.151:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -251,9 +251,10 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-+VDD_LED: supplied by +VDD_MUX
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
++VDD_LED: supplied by +VDD_MUX
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
@@ -272,9 +273,6 @@
 Freeing unused kernel memory: 960K (ffff800000c00000 - ffff800000cf0000)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
 
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:07.074439105 -0800
+++ /tmp/foob	2016-11-21 03:44:07.074439105 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -47,20 +47,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 7286552 (6f2f18 hex)
+Bytes transferred = 7286920 (6f3088 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6f2f18 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6f3088 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc5-128069-ga25f094 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 14 02:04:24 PST 2016
+Linux version 4.9.0-rc6-128439-g9c76358 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 21 03:34:34 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c1427f80, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c1427fc0, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -101,7 +101,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1429a60   (1191 kB)
+      .data : 0xc1300000 - 0xc1429aa0   (1191 kB)
        .bss : 0xc142b000 - 0xc1480fc8   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -326,29 +326,28 @@
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 mmc0: host does not support reading read-only switch, assuming write-enable
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
-hub 2-0:1.0: USB hub found
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
  mmcblk0: p1
-hub 2-0:1.0: 1 port detected
 mmc1: new high speed SD card at address b368
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+hub 2-0:1.0: USB hub found
+hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-05 23:21:00 UTC (1423178460)
+rtc-em3027 2-0056: setting system clock to 2015-02-05 23:45:56 UTC (1423179956)
  mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-random: fast init done
-asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
+random: fast init done
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+root@(none):/# [    3.086772] asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:07.082439135 -0800
+++ /tmp/foob	2016-11-21 03:44:07.082439135 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################################
-	 4.1 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5453296 (5335f0 hex)
+Bytes transferred = 5454800 (533bd0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5335f0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x533bd0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc5-128069-ga25f094 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 14 02:04:10 PST 2016
+Linux version 4.9.0-rc6-128439-g9c76358 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 21 03:34:33 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -138,7 +138,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.384295] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?[    0.380767] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 usb1_vbus: Failed to request enable GPIO170: -517
 reg-fixed-voltage regulators:regulator@2: Failed to register regulator: -517
@@ -306,17 +306,16 @@
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 NET: Registered protocol family 10
-mip6: Mobile IPv6
-sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
-NET: Registered protocol family 17
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
+mip6: Mobile IPv6
+ mmcblk0: p1
+sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 mmc1: new high speed SD card at address b368
-NET: Registered protocol family 15
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk0: p1
- mmcblk1: p1 p2
+NET: Registered protocol family 17
+NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
@@ -334,14 +333,15 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
+ mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-05 23:19:29 UTC (1423178369)
+rtc-em3027 2-0056: setting system clock to 2015-02-05 23:47:26 UTC (1423180046)
 ALSA device list:
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:07.098439199 -0800
+++ /tmp/foob	2016-11-21 03:44:07.098439199 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 741521 us
-Basic modules init took 350376 us
-Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14368 us
-Carveout took 252746 us
-CPU initialization took 92901 us
-Total time taken by TegraBoot 710391 us
+Tegraboot started after 722955 us
+Basic modules init took 348878 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14363 us
+Carveout took 252771 us
+CPU initialization took 92312 us
+Total time taken by TegraBoot 708324 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.583] LPDDR4 Training: Number of tables = 10
+64b[0001.562] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-14445056 bytes read in 784 ms (17.6 MiB/s)
+14445056 bytes read in 783 ms (17.6 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 43755 bytes read in 453 ms (93.8 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 502 ms (3.1 MiB/s)
+1627787 bytes read in 503 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc5-152502-ga25f094 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Nov 14 02:07:04 PST 2016
+Linux version 4.9.0-rc6-152872-g9c76358 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Nov 21 03:37:03 PST 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -266,7 +266,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.531:1): initialized
+audit: type=2000 audit(0.523:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -347,6 +347,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (ffff800000c70000 - ffff800000d70000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job contr[    1.453209] random: fast init done
+bash: no job contr[    1.435753] random: fast init done
 ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:07.118439279 -0800
+++ /tmp/foob	2016-11-21 03:44:07.118439279 -0800
@@ -1,4 +1,5 @@
-0.000000] efi: Getting EFI parameters from FDT:
+]
+efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 16 MiB at 0x00000000f5400000
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
@@ -59,7 +60,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x5068: expected 0x2004f400
+boot misc3 0x51b8: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -72,7 +73,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc5-152502-ga25f094 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Nov 14 02:07:04 PST 2016
+Linux version 4.9.0-rc6-152872-g9c76358 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Nov 21 03:37:03 PST 2016
 Boot CPU: AArch64 Processor [411fd071]
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -135,7 +136,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x5068: expected 0x2004f400
+boot misc3 0x51b8: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -222,7 +223,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.363:1): initialized
+audit: type=2000 audit(0.355:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -252,6 +253,7 @@
 PP1100_SOC_RTC: Bringing 1000000uV into 850000-850000uV
 PP2800L_FCAM: Bringing 3100000uV into 2800000-2800000uV
 PP_LDO6: Bringing 2800000uV into 1800000-1800000uV
+random: fast init done
 hisi_sas: driver version v1.6
 libphy: Fixed MDIO Bus: probed
 tun: Universal TUN/TAP device driver, 1.6
@@ -287,7 +289,6 @@
 hub 2-0:1.0: 4 ports detected
 usbcore: registered new interface driver usb-storage
 mousedev: PS/2 mouse device common for all mice
-random: fast init done
 max77686-rtc max77620-rtc: rtc core: registered max77620-rtc as rtc0
 tegra_rtc 7000e000.rtc: rtc core: registered 7000e000.rtc as rtc1
 tegra_rtc 7000e000.rtc: Tegra internal Real Time Clock
@@ -312,12 +313,12 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
-max77686-rtc max77620-rtc: setting system clock to 2000-08-25 15:36:07 UTC (967217767)
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+max77686-rtc max77620-rtc: setting system clock to 2000-09-01 17:06:00 UTC (967827960)
 PP1200_AVDD: disabling
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
 PP2800L_FCAM: disabling
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:07.126439310 -0800
+++ /tmp/foob	2016-11-21 03:44:07.126439310 -0800
@@ -39,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 7286552 (6f2f18 hex)
+Bytes transferred = 7286920 (6f3088 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 51572 (c974 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6f2f18 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6f3088 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff973 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc5-128069-ga25f094 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 14 02:04:24 PST 2016
+Linux version 4.9.0-rc6-128439-g9c76358 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 21 03:34:34 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -70,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1427f80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1427fc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -93,7 +93,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1429a60   (1191 kB)
+      .data : 0xc1300000 - 0xc1429aa0   (1191 kB)
        .bss : 0xc142b000 - 0xc1480fc8   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -304,8 +304,6 @@
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: 2x3 configuration
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
@@ -314,8 +312,10 @@
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
- mmcblk0: p1 p2 p3
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
+ mmcblk0: p1 p2 p3
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -381,9 +381,9 @@
 EXT4-fs (mmcblk1p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.908605] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.903591] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-21 03:44:07.134439342 -0800
+++ /tmp/foob	2016-11-21 03:44:07.138439358 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################################
-	 3.4 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 5453296 (5335f0 hex)
+Bytes transferred = 5454800 (533bd0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51572 (c974 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5335f0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x533bd0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff973 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc5-128069-ga25f094 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 14 02:04:10 PST 2016
+Linux version 4.9.0-rc6-128439-g9c76358 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 21 03:34:33 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -130,7 +130,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.571857] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?[    0.569024] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 tegra-mc 7000f000.memory-controller: no memory timings for RAM code 3 registered
 vdd_5v_sata: Failed to request enable GPIO30: -517
@@ -381,4 +381,4 @@
 Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.501006] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.500831] usb 2-1: new high-speed USB device number 2 using tegra-ehci
