Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-04-03 07:41:17.051147361 -0700
+++ /tmp/foob	2016-04-03 07:41:17.055147360 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Mar 30 01:36:39 PDT 2016
+Linux version 4.6.0-rc2+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Apr 3 07:36:00 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -9,7 +9,7 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: Trusted OS migration not required
-percpu: Embedded 20 pages/cpu @ffffffc03efd4000 s43008 r8192 d30720 u81920
+percpu: Embedded 20 pages/cpu @ffffffc03efd3000 s43008 r8192 d30720 u81920
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -19,14 +19,14 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 936628K/1048576K available (6840K kernel code, 615K rwdata, 2880K rodata, 812K init, 241K bss, 95564K reserved, 16384K cma-reserved)
+Memory: 936376K/1048576K available (6996K kernel code, 625K rwdata, 2956K rodata, 816K init, 244K bss, 95816K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbdbfff0000   (   246 GB)
-      .text : 0xffffff8008080000 - 0xffffff800872c000   (  6832 KB)
-    .rodata : 0xffffff800872c000 - 0xffffff8008a00000   (  2896 KB)
-      .init : 0xffffff8008a00000 - 0xffffff8008acb000   (   812 KB)
-      .data : 0xffffff8008acb000 - 0xffffff8008b64e00   (   616 KB)
+      .text : 0xffffff8008080000 - 0xffffff8008753000   (  6988 KB)
+    .rodata : 0xffffff8008753000 - 0xffffff8008a3a000   (  2972 KB)
+      .init : 0xffffff8008a3a000 - 0xffffff8008b06000   (   816 KB)
+      .data : 0xffffff8008b06000 - 0xffffff8008ba2400   (   625 KB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
               0xffffffbdc0000000 - 0xffffffbdc1000000   (    16 MB actual)
     fixed   : 0xffffffbffe7fd000 - 0xffffffbffec00000   (  4108 KB)
@@ -60,7 +60,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffff8008732000, 1 data @ ffffff8008ad0000)
+vdso: 2 pages (1 code @ ffffff8008759000, 1 data @ ffffff8008b0c000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -94,7 +94,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.512:1): initialized
+audit: type=2000 audit(0.532:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -154,7 +154,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-03-30 08:37:10 UTC (1459327030)
+rtc-pl031 9010000.pl031: setting system clock to 2016-04-03 14:36:31 UTC (1459694191)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
@@ -163,7 +163,7 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 812K (ffffff8008a00000 - ffffff8008acb000)
+Freeing unused kernel memory: 816K (ffffff8008a3a000 - ffffff8008b06000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-03 07:41:17.095147343 -0700
+++ /tmp/foob	2016-04-03 07:41:17.095147343 -0700
@@ -41,18 +41,18 @@
 	 #################
 	 3.4 MiB/s
 done
-Bytes transferred = 6915288 (6984d8 hex)
+Bytes transferred = 6918256 (699070 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6984d8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x699070 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-81957-gf55532a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Mar 30 01:35:14 PDT 2016
+Linux version 4.6.0-rc2-82191-g9735a22 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Apr 3 07:34:13 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141b440, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141b3c0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -90,9 +90,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc104fa58   (14623 kB)
+      .text : 0xc0208000 - 0xc104fa5c   (14623 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141d480   (1142 kB)
+      .data : 0xc1300000 - 0xc141d400   (1141 kB)
        .bss : 0xc141f000 - 0xc1474558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -276,9 +276,9 @@
 vdd-sys-5v0: supplied by vdd_ac_bat
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
 vdd_hdmi_5v0: supplied by vdd-sys-5v0
+mmc0: new high speed SDHC card at address aaaa
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
  mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
@@ -298,7 +298,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-22 19:54:55 UTC (948570895)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-27 01:54:24 UTC (948938064)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -310,4 +310,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.229377] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.234481] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-03 07:41:17.107147339 -0700
+++ /tmp/foob	2016-04-03 07:41:17.111147337 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5312328 (510f48 hex)
+Bytes transferred = 5313296 (511310 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x510f48 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x511310 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-81957-gf55532a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Mar 30 01:34:55 PDT 2016
+Linux version 4.6.0-rc2-82191-g9735a22 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Apr 3 07:34:07 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2224   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab222c   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a04   ( 363 kB)
@@ -249,15 +249,15 @@
 mmc1: SDHCI controller on 78000600.sdhci [78000600.sdhci] using ADMA
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmc1: new high speed MMC card at address 0001
-mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
-mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
-mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
-mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
- mmcblk0: p1 p2 p3
+mmcblk0: mmc1:0001 SEM32G 29.1 GiB 
+mmcblk0boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+mmc0: new high speed SDHC card at address aaaa
+mmcblk1: mmc0:aaaa SL16G 14.8 GiB 
+mmcblk0boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
+mmcblk0rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+ mmcblk1: p1 p2 p3
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
@@ -295,16 +295,16 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-22 19:52:25 UTC (948570745)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-27 01:51:48 UTC (948937908)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
 ALSA device list:
   #0: NVIDIA Tegra Dalmore
-EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
-EXT4-fs (mmcblk1p1): recovery complete
-EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+EXT4-fs (mmcblk0p1): recovery complete
+EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-03 07:41:17.115147336 -0700
+++ /tmp/foob	2016-04-03 07:41:17.119147334 -0700
@@ -43,18 +43,18 @@
 	 #################
 	 3.2 MiB/s
 done
-Bytes transferred = 6915288 (6984d8 hex)
+Bytes transferred = 6918256 (699070 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6984d8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x699070 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-81957-gf55532a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Mar 30 01:35:14 PDT 2016
+Linux version 4.6.0-rc2-82191-g9735a22 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Apr 3 07:34:13 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141b440, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141b3c0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -92,9 +92,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc104fa58   (14623 kB)
+      .text : 0xc0208000 - 0xc104fa5c   (14623 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141d480   (1142 kB)
+      .data : 0xc1300000 - 0xc141d400   (1141 kB)
        .bss : 0xc141f000 - 0xc1474558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -374,7 +374,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:20 UTC (946684940)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:09 UTC (946684929)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-03 07:41:17.135147328 -0700
+++ /tmp/foob	2016-04-03 07:41:17.139147326 -0700
@@ -39,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5312328 (510f48 hex)
+Bytes transferred = 5313296 (511310 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x510f48 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x511310 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -61,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-81957-gf55532a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Mar 30 01:34:55 PDT 2016
+Linux version 4.6.0-rc2-82191-g9735a22 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Apr 3 07:34:07 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -92,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2224   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab222c   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a04   ( 363 kB)
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-04-03 07:41:17.143147325 -0700
+++ /tmp/foob	2016-04-03 07:41:17.147147323 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800236 us
+Bootloader-Cpu Init at (time stamp): 1800251 us
 
 Pinmux changes applied in kernel way
 
@@ -62,29 +62,29 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800210  us
-Bootloader (AosInit) time: 733442 us
-Bootloader Main Init time: 180286 us
-BlockDevMgrInit time: 157538 us
+Miscellaneous avp time: 1800225  us
+Bootloader (AosInit) time: 733456 us
+Bootloader Main Init time: 180323 us
+BlockDevMgrInit time: 157557 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
-FsMgrInit time: 2 us
-Partition Table load: 77093 us 
-Display init time: 574392 us
-Kernel read time: 301 msec
-Load Kernel Size:   13053952 Bytes
-Bl Kernel ReadSpeed: 43368000 Bytes/sec
-Pre-Kernel Jump time: 1488631 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5492650 us
+StorMgrInit time: 30 us
+FsMgrInit time: 3 us
+Partition Table load: 77069 us 
+Display init time: 574409 us
+Kernel read time: 308 msec
+Load Kernel Size:   13305856 Bytes
+Bl Kernel ReadSpeed: 43200000 Bytes/sec
+Pre-Kernel Jump time: 1507764 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5520663 us
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Mar 30 01:36:39 PDT 2016
+Linux version 4.6.0-rc2+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Apr 3 07:36:00 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 16 MiB at 0x00000000eb400000
 /cpus/cpu@1: missing enable-method property
-percpu: Embedded 20 pages/cpu @ffffffc06c5a4000 s43008 r8192 d30720 u81920
+percpu: Embedded 20 pages/cpu @ffffffc06c5a3000 s43008 r8192 d30720 u81920
 Detected PIPT I-cache on CPU0
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 436968
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=94207 modem_id=0 android.kerneltype=normal
@@ -92,14 +92,14 @@
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
 software IO TLB [mem 0xe5200000-0xe9200000] (64MB) mapped at [ffffffc065200000-ffffffc0691fffff]
-Memory: 1643996K/1775616K available (6840K kernel code, 615K rwdata, 2880K rodata, 812K init, 241K bss, 115236K reserved, 16384K cma-reserved)
+Memory: 1643744K/1775616K available (6996K kernel code, 625K rwdata, 2956K rodata, 816K init, 244K bss, 115488K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbdbfff0000   (   246 GB)
-      .text : 0xffffff8008080000 - 0xffffff800872c000   (  6832 KB)
-    .rodata : 0xffffff800872c000 - 0xffffff8008a00000   (  2896 KB)
-      .init : 0xffffff8008a00000 - 0xffffff8008acb000   (   812 KB)
-      .data : 0xffffff8008acb000 - 0xffffff8008b64e00   (   616 KB)
+      .text : 0xffffff8008080000 - 0xffffff8008753000   (  6988 KB)
+    .rodata : 0xffffff8008753000 - 0xffffff8008a3a000   (  2972 KB)
+      .init : 0xffffff8008a3a000 - 0xffffff8008b06000   (   816 KB)
+      .data : 0xffffff8008b06000 - 0xffffff8008ba2400   (   625 KB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
               0xffffffbdc0000000 - 0xffffffbdc1b18000   (    27 MB actual)
     fixed   : 0xffffffbffe7fd000 - 0xffffffbffec00000   (  4108 KB)
@@ -133,7 +133,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffff8008732000, 1 data @ ffffff8008ad0000)
+vdso: 2 pages (1 code @ ffffff8008759000, 1 data @ ffffff8008b0c000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -179,7 +179,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.823:1): initialized
+audit: type=2000 audit(0.847:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -242,8 +242,8 @@
 mmc1: Unknown controller version (3). You may experience problems.
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
-mmc0: new HS200 MMC card at address 0001
 hctosys: unable to open rtc device (rtc0)
+mmc0: new HS200 MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
@@ -257,7 +257,7 @@
 +5V_HDMI_CON: disabling
 ALSA device list:
   No soundcards found.
-Freeing unused kernel memory: 812K (ffffff8008a00000 - ffffff8008acb000)
+Freeing unused kernel memory: 816K (ffffff8008a3a000 - ffffff8008b06000)
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-03 07:41:17.151147322 -0700
+++ /tmp/foob	2016-04-03 07:41:17.155147320 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -47,20 +47,20 @@
 	 #################################################################
 	 #################################################################
 	 #################
-	 903.3 KiB/s
+	 906.3 KiB/s
 done
-Bytes transferred = 6915288 (6984d8 hex)
+Bytes transferred = 6918256 (699070 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 710.9 KiB/s
+	 896.5 KiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6984d8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x699070 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-81957-gf55532a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Mar 30 01:35:14 PDT 2016
+Linux version 4.6.0-rc2-82191-g9735a22 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Apr 3 07:34:13 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c141b440, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c141b3c0, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -97,9 +97,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc104fa58   (14623 kB)
+      .text : 0xc0208000 - 0xc104fa5c   (14623 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141d480   (1142 kB)
+      .data : 0xc1300000 - 0xc141d400   (1141 kB)
        .bss : 0xc141f000 - 0xc1474558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -325,7 +325,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 13:13:38 UTC (1419772418)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 13:37:54 UTC (1419773874)
 Waiting for root device PARTUUID=11eb23c1-01...
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
@@ -333,12 +333,11 @@
  mmcblk0: p1
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
-hub 1-1:1.0: USB hub found
  mmcblk1: p1 p2
+hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-03 07:41:17.167147315 -0700
+++ /tmp/foob	2016-04-03 07:41:17.171147312 -0700
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 4.1 MiB/s
+	 917 KiB/s
 done
-Bytes transferred = 5312328 (510f48 hex)
+Bytes transferred = 5313296 (511310 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 896.5 KiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x510f48 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x511310 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-81957-gf55532a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Mar 30 01:34:55 PDT 2016
+Linux version 4.6.0-rc2-82191-g9735a22 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Apr 3 07:34:07 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2224   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab222c   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a04   ( 363 kB)
@@ -304,17 +304,20 @@
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
-NET: Registered protocol family 10
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
-mmc0: host does not support reading read-only switch, assuming write-enable
-mmc0: new high speed SDHC card at address b368
-mmc1: new high speed SD card at address b368
+NET: Registered protocol family 10
 mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
+mmc0: host does not support reading read-only switch, assuming write-enable
 NET: Registered protocol family 17
+mmc1: new high speed SD card at address b368
+mmc0: new high speed SDHC card at address b368
+mmcblk1: mmc0:b368 NCard 3.70 GiB 
+ mmcblk1: p1
+mmcblk0: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk0: p1 p2
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
@@ -330,24 +333,21 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
- mmcblk0: p1
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
+hub 1-1:1.0: USB hub found
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 13:12:09 UTC (1419772329)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 13:36:29 UTC (1419773789)
 ALSA device list:
   #0: tegra-trimslice
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-EXT4-fs (mmcblk0p1): recovery complete
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+EXT4-fs (mmcblk1p1): recovery complete
+EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-04-03 07:41:17.179147311 -0700
+++ /tmp/foob	2016-04-03 07:41:17.179147311 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 723818 us
-Basic modules init took 350458 us
+Tegraboot started after 735139 us
+Basic modules init took 348797 us
 Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14365 us
-Carveout took 252730 us
-CPU initialization took 93158 us
-Total time taken by TegraBoot 710711 us
+Next stage binary read took 14373 us
+Carveout took 253011 us
+CPU initialization took 93109 us
+Total time taken by TegraBoot 709290 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.566] LPDDR4 Training: Number of tables = 10
+64b[0001.575] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=/dev/mmcblk0p1 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-11423232 bytes read in 713 ms (15.3 MiB/s)
+11674624 bytes read in 736 ms (15.1 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 37136 bytes read in 454 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 496 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Mar 30 01:36:39 PDT 2016
+Linux version 4.6.0-rc2+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Apr 3 07:36:00 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -149,7 +149,7 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_TYPE not supported.
-percpu: Embedded 20 pages/cpu @ffffffc0bff5b000 s43008 r8192 d30720 u81920
+percpu: Embedded 20 pages/cpu @ffffffc0bff5a000 s43008 r8192 d30720 u81920
 pcpu-alloc: s43008 r8192 d30720 u81920 alloc=20*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Detected PIPT I-cache on CPU0
@@ -166,14 +166,14 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffffffc07a000000-ffffffc07dffffff]
-Memory: 2981568K/3131392K available (6840K kernel code, 615K rwdata, 2880K rodata, 812K init, 241K bss, 133440K reserved, 16384K cma-reserved)
+Memory: 2981316K/3131392K available (6996K kernel code, 625K rwdata, 2956K rodata, 816K init, 244K bss, 133692K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbdbfff0000   (   246 GB)
-      .text : 0xffffff8008080000 - 0xffffff800872c000   (  6832 KB)
-    .rodata : 0xffffff800872c000 - 0xffffff8008a00000   (  2896 KB)
-      .init : 0xffffff8008a00000 - 0xffffff8008acb000   (   812 KB)
-      .data : 0xffffff8008acb000 - 0xffffff8008b64e00   (   616 KB)
+      .text : 0xffffff8008080000 - 0xffffff8008753000   (  6988 KB)
+    .rodata : 0xffffff8008753000 - 0xffffff8008a3a000   (  2972 KB)
+      .init : 0xffffff8008a3a000 - 0xffffff8008b06000   (   816 KB)
+      .data : 0xffffff8008b06000 - 0xffffff8008ba2400   (   625 KB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
               0xffffffbdc0000000 - 0xffffffbdc3000000   (    48 MB actual)
     fixed   : 0xffffffbffe7fd000 - 0xffffffbffec00000   (  4108 KB)
@@ -222,7 +222,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffff8008732000, 1 data @ ffffff8008ad0000)
+vdso: 2 pages (1 code @ ffffff8008759000, 1 data @ ffffff8008b0c000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -259,7 +259,7 @@
 kvm [1]: timer IRQ3
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.363:1): initialized
+audit: type=2000 audit(0.355:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -330,7 +330,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 812K (ffffff8008a00000 - ffffff8008acb000)
+Freeing unused kernel memory: 816K (ffffff8008a3a000 - ffffff8008b06000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-03 07:41:17.187147308 -0700
+++ /tmp/foob	2016-04-03 07:41:17.187147308 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #################
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 6915288 (6984d8 hex)
+Bytes transferred = 6918256 (699070 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6984d8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x699070 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-81957-gf55532a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Mar 30 01:35:14 PDT 2016
+Linux version 4.6.0-rc2-82191-g9735a22 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Apr 3 07:34:13 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141b440, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141b3c0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,9 +89,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc104fa58   (14623 kB)
+      .text : 0xc0208000 - 0xc104fa5c   (14623 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141d480   (1142 kB)
+      .data : 0xc1300000 - 0xc141d400   (1141 kB)
        .bss : 0xc141f000 - 0xc1474558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -362,4 +362,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.733959] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.728977] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-03 07:41:17.195147304 -0700
+++ /tmp/foob	2016-04-03 07:41:17.199147303 -0700
@@ -39,18 +39,18 @@
 	 #####################################
 	 3.3 MiB/s
 done
-Bytes transferred = 5312328 (510f48 hex)
+Bytes transferred = 5313296 (511310 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x510f48 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x511310 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-81957-gf55532a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Mar 30 01:34:55 PDT 2016
+Linux version 4.6.0-rc2-82191-g9735a22 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Apr 3 07:34:07 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2224   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab222c   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a04   ( 363 kB)
@@ -360,6 +360,7 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.980638] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
