Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-07-12 16:07:18.728151398 -0700
+++ /tmp/foob	2015-07-12 16:07:18.732151395 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.2.0-rc1+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Jul 5 12:35:15 PDT 2015
+Linux version 4.2.0-rc2+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Jul 12 16:04:39 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -19,7 +19,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 939204K/1048576K available (5509K kernel code, 442K rwdata, 2224K rodata, 452K init, 204K bss, 92988K reserved, 16384K cma-reserved)
+Memory: 939204K/1048576K available (5513K kernel code, 439K rwdata, 2224K rodata, 452K init, 203K bss, 92988K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -28,9 +28,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc000810000 - 0xffffffc000881000   (   452 KB)
-      .text : 0xffffffc000080000 - 0xffffffc00080f5f4   (  7742 KB)
-      .data : 0xffffffc000887000 - 0xffffffc0008f5800   (   442 KB)
+      .init : 0xffffffc000811000 - 0xffffffc000882000   (   452 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0008105f4   (  7746 KB)
+      .data : 0xffffffc000888000 - 0xffffffc0008f5c00   (   439 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -89,7 +89,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.420:1): initialized
+audit: type=2000 audit(0.380:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
@@ -136,8 +136,8 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 452K (ffffffc000810000 - ffffffc000881000)
-Freeing alternatives memory: 12K (ffffffc000881000 - ffffffc000884000)
+Freeing unused kernel memory: 452K (ffffffc000811000 - ffffffc000882000)
+Freeing alternatives memory: 12K (ffffffc000882000 - ffffffc000885000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-12 16:07:18.744151386 -0700
+++ /tmp/foob	2015-07-12 16:07:18.744151386 -0700
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6152088 (5ddf98 hex)
+Bytes transferred = 6153392 (5de4b0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5ddf98 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5de4b0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc1-26550-gd770e55 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jul 5 12:33:57 PDT 2015
+Linux version 4.2.0-rc2-26810-gbc0195a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jul 12 16:03:09 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -88,7 +88,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d0f9d4   (11295 kB)
+      .text : 0xc0208000 - 0xc0d0fa60   (11295 kB)
       .init : 0xc0d10000 - 0xc0dfc000   ( 944 kB)
       .data : 0xc0dfc000 - 0xc0effa80   (1039 kB)
        .bss : 0xc0f02000 - 0xc0f53c28   ( 328 kB)
@@ -296,7 +296,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-16 04:25:40 UTC (947996740)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-23 07:55:13 UTC (948614113)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -308,4 +308,4 @@
 Freeing unused kernel memory: 944K (c0d10000 - c0dfc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.193848] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.193892] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-12 16:07:18.756151378 -0700
+++ /tmp/foob	2015-07-12 16:07:18.760151375 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 #####################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5066080 (4d4d60 hex)
+Bytes transferred = 5066936 (4d50b8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4d4d60 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4d50b8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc1-26550-gd770e55 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jul 5 12:34:32 PDT 2015
+Linux version 4.2.0-rc2-26810-gbc0195a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jul 12 16:03:46 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0921858   (9319 kB)
+      .text : 0xc0008000 - 0xc09218b0   (9319 kB)
       .init : 0xc0922000 - 0xc0970000   ( 312 kB)
       .data : 0xc0970000 - 0xc09c6880   ( 347 kB)
        .bss : 0xc09c9000 - 0xc0a23c54   ( 364 kB)
@@ -295,7 +295,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-16 04:26:18 UTC (947996778)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-23 07:55:53 UTC (948614153)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -310,4 +310,4 @@
 Freeing unused kernel memory: 312K (c0922000 - c0970000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.739994] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.740031] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-12 16:07:18.768151368 -0700
+++ /tmp/foob	2015-07-12 16:07:18.768151368 -0700
@@ -42,18 +42,18 @@
 	 ##############################
 	 3.2 MiB/s
 done
-Bytes transferred = 6152088 (5ddf98 hex)
+Bytes transferred = 6153392 (5de4b0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3.2 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 62999 (f617 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5ddf98 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5de4b0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012616
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc1-26550-gd770e55 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jul 5 12:33:57 PDT 2015
+Linux version 4.2.0-rc2-26810-gbc0195a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jul 12 16:03:09 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d0f9d4   (11295 kB)
+      .text : 0xc0208000 - 0xc0d0fa60   (11295 kB)
       .init : 0xc0d10000 - 0xc0dfc000   ( 944 kB)
       .data : 0xc0dfc000 - 0xc0effa80   (1039 kB)
        .bss : 0xc0f02000 - 0xc0f53c28   ( 328 kB)
@@ -357,10 +357,10 @@
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
 Freeing unused kernel memory: 944K (c0d10000 - c0dfc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-12 16:07:18.776151363 -0700
+++ /tmp/foob	2015-07-12 16:07:18.780151360 -0700
@@ -41,18 +41,18 @@
 	 #####################
 	 3.2 MiB/s
 done
-Bytes transferred = 5066080 (4d4d60 hex)
+Bytes transferred = 5066936 (4d50b8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 62999 (f617 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4d4d60 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4d50b8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012616
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc1-26550-gd770e55 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jul 5 12:34:32 PDT 2015
+Linux version 4.2.0-rc2-26810-gbc0195a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jul 12 16:03:46 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -92,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0921858   (9319 kB)
+      .text : 0xc0008000 - 0xc09218b0   (9319 kB)
       .init : 0xc0922000 - 0xc0970000   ( 312 kB)
       .data : 0xc0970000 - 0xc09c6880   ( 347 kB)
        .bss : 0xc09c9000 - 0xc0a23c54   ( 364 kB)
@@ -360,8 +360,8 @@
   #1: NVIDIA Tegra Jetson TK1
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT3-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (240)
-EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (244)
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
+EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (244)
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-12 16:07:18.784151357 -0700
+++ /tmp/foob	2015-07-12 16:07:18.788151354 -0700
@@ -48,18 +48,18 @@
 	 ##############################
 	 4.1 MiB/s
 done
-Bytes transferred = 6152088 (5ddf98 hex)
+Bytes transferred = 6153392 (5de4b0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5ddf98 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5de4b0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc1-26550-gd770e55 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jul 5 12:33:57 PDT 2015
+Linux version 4.2.0-rc2-26810-gbc0195a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jul 12 16:03:09 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -96,7 +96,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d0f9d4   (11295 kB)
+      .text : 0xc0208000 - 0xc0d0fa60   (11295 kB)
       .init : 0xc0d10000 - 0xc0dfc000   ( 944 kB)
       .data : 0xc0dfc000 - 0xc0effa80   (1039 kB)
        .bss : 0xc0f02000 - 0xc0f53c28   ( 328 kB)
@@ -324,30 +324,29 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 mmc0: host does not support reading read-only switch, assuming write-enable
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 17:33:07 UTC (1419615187)
-Waiting for root device /dev/mmcblk0p1...
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
+rtc-em3027 2-0056: setting system clock to 2014-12-26 18:59:54 UTC (1419620394)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 944K (c0d10000 - c0dfc000)
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
- mmcblk1: p1 p2
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-12 16:07:18.792151351 -0700
+++ /tmp/foob	2015-07-12 16:07:18.796151348 -0700
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 #####################
-	 4 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 5066080 (4d4d60 hex)
+Bytes transferred = 5066936 (4d50b8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4d4d60 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4d50b8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc1-26550-gd770e55 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jul 5 12:34:32 PDT 2015
+Linux version 4.2.0-rc2-26810-gbc0195a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jul 12 16:03:46 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -99,7 +99,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0921858   (9319 kB)
+      .text : 0xc0008000 - 0xc09218b0   (9319 kB)
       .init : 0xc0922000 - 0xc0970000   ( 312 kB)
       .data : 0xc0970000 - 0xc09c6880   ( 347 kB)
        .bss : 0xc09c9000 - 0xc0a23c54   ( 364 kB)
@@ -344,16 +344,16 @@
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
+mmc1: new high speed SD card at address b368
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-mmc1: new high speed SD card at address b368
-input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
-rtc-em3027 2-0056: setting system clock to 2014-12-26 17:33:26 UTC (1419615206)
+ mmcblk1: p1 p2
+input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+rtc-em3027 2-0056: setting system clock to 2014-12-26 19:00:13 UTC (1419620413)
 ALSA device list:
   #0: tegra-trimslice
- mmcblk1: p1 p2
 kjournald starting.  Commit interval 5 seconds
 EXT3-fs (mmcblk0p1): using internal journal
 EXT3-fs (mmcblk0p1): recovery complete
@@ -361,7 +361,7 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 312K (c0922000 - c0970000)
-bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+bash: cannot set[    2.242201] usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+ terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-12 16:07:18.804151342 -0700
+++ /tmp/foob	2015-07-12 16:07:18.804151342 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.4 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 6152088 (5ddf98 hex)
+Bytes transferred = 6153392 (5de4b0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5ddf98 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5de4b0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc1-26550-gd770e55 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jul 5 12:33:57 PDT 2015
+Linux version 4.2.0-rc2-26810-gbc0195a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jul 12 16:03:09 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -88,7 +88,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d0f9d4   (11295 kB)
+      .text : 0xc0208000 - 0xc0d0fa60   (11295 kB)
       .init : 0xc0d10000 - 0xc0dfc000   ( 944 kB)
       .data : 0xc0dfc000 - 0xc0effa80   (1039 kB)
        .bss : 0xc0f02000 - 0xc0f53c28   ( 328 kB)
@@ -368,4 +368,4 @@
 Freeing unused kernel memory: 944K (c0d10000 - c0dfc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.783382] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.768770] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-12 16:07:18.812151336 -0700
+++ /tmp/foob	2015-07-12 16:07:18.816151333 -0700
@@ -39,18 +39,18 @@
 	 #####################
 	 3.4 MiB/s
 done
-Bytes transferred = 5066080 (4d4d60 hex)
+Bytes transferred = 5066936 (4d50b8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4d4d60 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4d50b8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc1-26550-gd770e55 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jul 5 12:34:32 PDT 2015
+Linux version 4.2.0-rc2-26810-gbc0195a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jul 12 16:03:46 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0921858   (9319 kB)
+      .text : 0xc0008000 - 0xc09218b0   (9319 kB)
       .init : 0xc0922000 - 0xc0970000   ( 312 kB)
       .data : 0xc0970000 - 0xc09c6880   ( 347 kB)
        .bss : 0xc09c9000 - 0xc0a23c54   ( 364 kB)
@@ -372,4 +372,4 @@
 Freeing unused kernel memory: 312K (c0922000 - c0970000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.436734] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.436380] usb 2-1: new high-speed USB device number 2 using tegra-ehci
