Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-05-10 16:39:09.355684290 -0700
+++ /tmp/foob	2015-05-10 16:39:09.359684288 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.1.0-rc2+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun May 3 19:34:17 PDT 2015
+Linux version 4.1.0-rc3+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun May 10 16:35:40 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -87,7 +87,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.440:1): initialized
+audit: type=2000 audit(0.460:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-05-10 16:39:09.367684284 -0700
+++ /tmp/foob	2015-05-10 16:39:09.367684284 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ###################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6227592 (5f0688 hex)
+Bytes transferred = 6238920 (5f32c8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5f0688 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5f32c8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-rc2-12405-g5ebe6af (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 3 19:33:16 PDT 2015
+Linux version 4.1.0-rc3-12646-g030bbdb (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 10 16:34:08 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f1d340, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f25b80, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999276K/2096128K available (7987K kernel code, 1020K rwdata, 3576K rodata, 816K init, 317K bss, 31316K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999244K/2096128K available (7987K kernel code, 1022K rwdata, 3588K rodata, 836K init, 318K bss, 31348K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d53f54   (11568 kB)
-      .init : 0xc0d54000 - 0xc0e20000   ( 816 kB)
-      .data : 0xc0e20000 - 0xc0f1f320   (1021 kB)
-       .bss : 0xc0f22000 - 0xc0f716a8   ( 318 kB)
+      .text : 0xc0208000 - 0xc0d56f60   (11580 kB)
+      .init : 0xc0d57000 - 0xc0e28000   ( 836 kB)
+      .data : 0xc0e28000 - 0xc0f27b60   (1023 kB)
+       .bss : 0xc0f2a000 - 0xc0f79a68   ( 319 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -303,7 +303,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-08 21:57:48 UTC (971042268)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-15 18:59:05 UTC (971636345)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -314,7 +314,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 816K (c0d54000 - c0e20000)
+Freeing unused kernel memory: 836K (c0d57000 - c0e28000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.628689] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.618684] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-05-10 16:39:09.375684279 -0700
+++ /tmp/foob	2015-05-10 16:39:09.379684277 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ########
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 4882864 (4a81b0 hex)
+Bytes transferred = 4883712 (4a8500 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 2.2 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4a81b0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4a8500 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-rc2-12405-g5ebe6af (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 3 19:33:31 PDT 2015
+Linux version 4.1.0-rc3-12646-g030bbdb (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 10 16:34:21 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -91,10 +91,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc08c7560   (8958 kB)
+      .text : 0xc0008000 - 0xc08c74fc   (8958 kB)
       .init : 0xc08c8000 - 0xc0916000   ( 312 kB)
       .data : 0xc0916000 - 0xc0963f20   ( 312 kB)
-       .bss : 0xc0966000 - 0xc09bde94   ( 352 kB)
+       .bss : 0xc0966000 - 0xc09bde14   ( 352 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
 NR_IRQS:16 nr_irqs:16 16
@@ -294,7 +294,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-08 21:58:19 UTC (971042299)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-15 18:59:31 UTC (971636371)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -309,4 +309,4 @@
 Freeing unused kernel memory: 312K (c08c8000 - c0916000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.779705] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.779651] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-05-10 16:39:09.387684272 -0700
+++ /tmp/foob	2015-05-10 16:39:09.391684270 -0700
@@ -33,17 +33,19 @@
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.multi_v7_defconfig'.
 Load address: 0x81000000
-Loading: *#################################################################
+Loading: *EHCI timed out on TD - token=0x88008d80
+T #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 ###################################
-	 3.1 MiB/s
+	 545.9 KiB/s
 done
-Bytes transferred = 6227592 (5f0688 hex)
+Bytes transferred = 6238920 (5f32c8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
@@ -53,7 +55,7 @@
 done
 Bytes transferred = 63003 (f61b hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5f0688 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5f32c8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 8201261a
@@ -61,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-rc2-12405-g5ebe6af (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 3 19:33:16 PDT 2015
+Linux version 4.1.0-rc3-12646-g030bbdb (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 10 16:34:08 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -70,7 +72,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f1d340, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f25b80, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -83,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999172K/2096128K available (7987K kernel code, 1020K rwdata, 3576K rodata, 816K init, 317K bss, 31420K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999140K/2096128K available (7987K kernel code, 1022K rwdata, 3588K rodata, 836K init, 318K bss, 31452K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,10 +93,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d53f54   (11568 kB)
-      .init : 0xc0d54000 - 0xc0e20000   ( 816 kB)
-      .data : 0xc0e20000 - 0xc0f1f320   (1021 kB)
-       .bss : 0xc0f22000 - 0xc0f716a8   ( 318 kB)
+      .text : 0xc0208000 - 0xc0d56f60   (11580 kB)
+      .init : 0xc0d57000 - 0xc0e28000   ( 836 kB)
+      .data : 0xc0e28000 - 0xc0f27b60   (1023 kB)
+       .bss : 0xc0f2a000 - 0xc0f79a68   ( 319 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -353,7 +355,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:18 UTC (949363218)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:27 UTC (949363227)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 +2.8V_RUN_CAM_AF: disabling
@@ -372,7 +374,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 816K (c0d54000 - c0e20000)
+Freeing unused kernel memory: 836K (c0d57000 - c0e28000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-05-10 16:39:09.403684264 -0700
+++ /tmp/foob	2015-05-10 16:39:09.403684264 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ########
-	 2.8 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 4882864 (4a81b0 hex)
+Bytes transferred = 4883712 (4a8500 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.7 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 63003 (f61b hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4a81b0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4a8500 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 8201261a
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-rc2-12405-g5ebe6af (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 3 19:33:31 PDT 2015
+Linux version 4.1.0-rc3-12646-g030bbdb (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 10 16:34:21 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -93,10 +93,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc08c7560   (8958 kB)
+      .text : 0xc0008000 - 0xc08c74fc   (8958 kB)
       .init : 0xc08c8000 - 0xc0916000   ( 312 kB)
       .data : 0xc0916000 - 0xc0963f20   ( 312 kB)
-       .bss : 0xc0966000 - 0xc09bde94   ( 352 kB)
+       .bss : 0xc0966000 - 0xc09bde14   ( 352 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
 NR_IRQS:16 nr_irqs:16 16
@@ -373,7 +373,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:49 UTC (949363249)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:58 UTC (949363258)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 +2.8V_RUN_CAM_AF: disabling
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-05-10 16:39:09.411684259 -0700
+++ /tmp/foob	2015-05-10 16:39:09.415684256 -0700
@@ -46,18 +46,18 @@
 	 ###################################
 	 4 MiB/s
 done
-Bytes transferred = 6227592 (5f0688 hex)
+Bytes transferred = 6238920 (5f32c8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5f0688 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5f32c8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -65,7 +65,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-rc2-12405-g5ebe6af (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 3 19:33:16 PDT 2015
+Linux version 4.1.0-rc3-12646-g030bbdb (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 10 16:34:08 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -74,7 +74,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0f1d340, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0f25b80, node_mem_map eeff8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -87,7 +87,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 960084K/1048576K available (7987K kernel code, 1020K rwdata, 3576K rodata, 816K init, 317K bss, 22956K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 960052K/1048576K available (7987K kernel code, 1022K rwdata, 3588K rodata, 836K init, 318K bss, 22988K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,10 +95,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d53f54   (11568 kB)
-      .init : 0xc0d54000 - 0xc0e20000   ( 816 kB)
-      .data : 0xc0e20000 - 0xc0f1f320   (1021 kB)
-       .bss : 0xc0f22000 - 0xc0f716a8   ( 318 kB)
+      .text : 0xc0208000 - 0xc0d56f60   (11580 kB)
+      .init : 0xc0d57000 - 0xc0e28000   ( 836 kB)
+      .data : 0xc0e28000 - 0xc0f27b60   (1023 kB)
+       .bss : 0xc0f2a000 - 0xc0f79a68   ( 319 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -170,7 +170,7 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 64
-CPU PMU: Failed to parse <no-node>/interrupt-affinity[0]
+CPU PMU: Failed to parse /pmu/interrupt-affinity[0]
 hw perfevents: enabled with armv7_cortex_a9 PMU driver, 7 counters available
 futex hash table entries: 512 (order: 3, 32768 bytes)
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
@@ -334,24 +334,24 @@
 can: netlink gateway (rev 20130117) max_hops=1
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
-mmc0: host does not support reading read-only switch, assuming write-enable
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 Registering SWP/SWPB emulation handler
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
+mmc0: host does not support reading read-only switch, assuming write-enable
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-hub 1-1:1.0: USB hub found
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-25 20:05:14 UTC (1419537914)
-hub 1-1:1.0: 4 ports detected
+hub 1-1:1.0: USB hub found
+rtc-em3027 2-0056: setting system clock to 2014-12-25 20:13:22 UTC (1419538402)
 ALSA device list:
   #0: tegra-trimslice
+hub 1-1:1.0: 4 ports detected
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
@@ -360,7 +360,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 816K (c0d54000 - c0e20000)
+Freeing unused kernel memory: 836K (c0d57000 - c0e28000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-05-10 16:39:09.419684254 -0700
+++ /tmp/foob	2015-05-10 16:39:09.423684252 -0700
@@ -43,20 +43,20 @@
 	 #################################################################
 	 #################################################################
 	 ########
-	 3.5 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 4882864 (4a81b0 hex)
+Bytes transferred = 4883712 (4a8500 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2.5 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4a81b0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4a8500 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-rc2-12405-g5ebe6af (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 3 19:33:31 PDT 2015
+Linux version 4.1.0-rc3-12646-g030bbdb (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 10 16:34:21 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -98,10 +98,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc08c7560   (8958 kB)
+      .text : 0xc0008000 - 0xc08c74fc   (8958 kB)
       .init : 0xc08c8000 - 0xc0916000   ( 312 kB)
       .data : 0xc0916000 - 0xc0963f20   ( 312 kB)
-       .bss : 0xc0966000 - 0xc09bde94   ( 352 kB)
+       .bss : 0xc0966000 - 0xc09bde14   ( 352 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -178,7 +178,7 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 64
-CPU PMU: Failed to parse <no-node>/interrupt-affinity[0]
+CPU PMU: Failed to parse /pmu/interrupt-affinity[0]
 hw perfevents: enabled with armv7_cortex_a9 PMU driver, 7 counters available
 futex hash table entries: 512 (order: 3, 32768 bytes)
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
@@ -342,7 +342,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-25 20:05:32 UTC (1419537932)
+rtc-em3027 2-0056: setting system clock to 2014-12-25 20:13:40 UTC (1419538420)
 hub 1-1:1.0: USB hub found
 ALSA device list:
   #0: tegra-trimslice
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-05-10 16:39:09.427684250 -0700
+++ /tmp/foob	2015-05-10 16:39:09.431684247 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ###################################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6227592 (5f0688 hex)
+Bytes transferred = 6238920 (5f32c8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51191 (c7f7 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5f0688 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5f32c8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff7f6 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-rc2-12405-g5ebe6af (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 3 19:33:16 PDT 2015
+Linux version 4.1.0-rc3-12646-g030bbdb (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 10 16:34:08 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f1d340, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f25b80, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999196K/2096128K available (7987K kernel code, 1020K rwdata, 3576K rodata, 816K init, 317K bss, 31396K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999164K/2096128K available (7987K kernel code, 1022K rwdata, 3588K rodata, 836K init, 318K bss, 31428K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d53f54   (11568 kB)
-      .init : 0xc0d54000 - 0xc0e20000   ( 816 kB)
-      .data : 0xc0e20000 - 0xc0f1f320   (1021 kB)
-       .bss : 0xc0f22000 - 0xc0f716a8   ( 318 kB)
+      .text : 0xc0208000 - 0xc0d56f60   (11580 kB)
+      .init : 0xc0d57000 - 0xc0e28000   ( 836 kB)
+      .data : 0xc0e28000 - 0xc0f27b60   (1023 kB)
+       .bss : 0xc0f2a000 - 0xc0f79a68   ( 319 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -173,7 +173,7 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 64
-CPU PMU: Failed to parse <no-node>/interrupt-affinity[0]
+CPU PMU: Failed to parse /pmu/interrupt-affinity[0]
 hw perfevents: enabled with armv7_cortex_a9 PMU driver, 7 counters available
 futex hash table entries: 1024 (order: 4, 65536 bytes)
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
@@ -375,7 +375,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 816K (c0d54000 - c0e20000)
+Freeing unused kernel memory: 836K (c0d57000 - c0e28000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.208883] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.223485] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-05-10 16:39:09.435684245 -0700
+++ /tmp/foob	2015-05-10 16:39:09.439684243 -0700
@@ -27,6 +27,7 @@
 Tegra30 (Beaver) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd'
 Tegra30 (Beaver) # setenv ipaddr 192.168.100.101; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra30 (Beaver) # tftpboot 0x81000000 zImage.tegra_defconfig
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'zImage.tegra_defconfig'.
@@ -37,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ########
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 4882864 (4a81b0 hex)
+Bytes transferred = 4883712 (4a8500 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51191 (c7f7 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4a81b0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4a8500 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff7f6 ... OK
@@ -61,7 +62,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-rc2-12405-g5ebe6af (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 3 19:33:31 PDT 2015
+Linux version 4.1.0-rc3-12646-g030bbdb (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 10 16:34:21 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -91,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc08c7560   (8958 kB)
+      .text : 0xc0008000 - 0xc08c74fc   (8958 kB)
       .init : 0xc08c8000 - 0xc0916000   ( 312 kB)
       .data : 0xc0916000 - 0xc0963f20   ( 312 kB)
-       .bss : 0xc0966000 - 0xc09bde94   ( 352 kB)
+       .bss : 0xc0966000 - 0xc09bde14   ( 352 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
 NR_IRQS:16 nr_irqs:16 16
@@ -178,7 +179,7 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 64
-CPU PMU: Failed to parse <no-node>/interrupt-affinity[0]
+CPU PMU: Failed to parse /pmu/interrupt-affinity[0]
 hw perfevents: enabled with armv7_cortex_a9 PMU driver, 7 counters available
 futex hash table entries: 1024 (order: 4, 65536 bytes)
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
@@ -371,4 +372,4 @@
 Freeing unused kernel memory: 312K (c08c8000 - c0916000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.496504] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.496489] usb 2-1: new high-speed USB device number 2 using tegra-ehci
