Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-08-09 13:37:08.914152168 -0700
+++ /tmp/foob	2015-08-09 13:37:08.918152167 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.2.0-rc5+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Aug 2 19:04:51 PDT 2015
+Linux version 4.2.0-rc6+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Aug 9 13:34:22 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -89,7 +89,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.400:1): initialized
+audit: type=2000 audit(0.420:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-09 13:37:08.922152166 -0700
+++ /tmp/foob	2015-08-09 13:37:08.926152163 -0700
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6156392 (5df068 hex)
+Bytes transferred = 6156160 (5def80 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5df068 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5def80 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-27916-g74d3329 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Aug 2 19:03:13 PDT 2015
+Linux version 4.2.0-rc6-28106-gf7644cb (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Aug 9 13:33:03 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -88,7 +88,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d0fa88   (11295 kB)
+      .text : 0xc0208000 - 0xc0d0faa0   (11295 kB)
       .init : 0xc0d10000 - 0xc0dfc000   ( 944 kB)
       .data : 0xc0dfc000 - 0xc0effb00   (1039 kB)
        .bss : 0xc0f02000 - 0xc0f53c28   ( 328 kB)
@@ -296,7 +296,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-13 10:56:20 UTC (950439380)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-20 05:26:22 UTC (951024382)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -308,4 +308,4 @@
 Freeing unused kernel memory: 944K (c0d10000 - c0dfc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.213877] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.193928] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-09 13:37:08.930152162 -0700
+++ /tmp/foob	2015-08-09 13:37:08.934152160 -0700
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 #####################
-	 3.2 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5067752 (4d53e8 hex)
+Bytes transferred = 5068016 (4d54f0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4d53e8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4d54f0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-27916-g74d3329 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Aug 2 19:03:46 PDT 2015
+Linux version 4.2.0-rc6-28106-gf7644cb (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Aug 9 13:33:28 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09c4ec0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09c6ec0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002668K/2096128K available (6945K kernel code, 346K rwdata, 2376K rodata, 308K init, 363K bss, 27924K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002660K/2096128K available (6945K kernel code, 346K rwdata, 2380K rodata, 312K init, 363K bss, 27932K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09228d0   (9323 kB)
-      .init : 0xc0923000 - 0xc0970000   ( 308 kB)
-      .data : 0xc0970000 - 0xc09c6a80   ( 347 kB)
-       .bss : 0xc09c9000 - 0xc0a23c58   ( 364 kB)
+      .text : 0xc0008000 - 0xc09238d4   (9327 kB)
+      .init : 0xc0924000 - 0xc0972000   ( 312 kB)
+      .data : 0xc0972000 - 0xc09c8a80   ( 347 kB)
+       .bss : 0xc09cb000 - 0xc0a25c58   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
 	Build-time adjustment of leaf fanout to 32.
@@ -294,7 +294,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-13 10:56:51 UTC (950439411)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-20 05:26:51 UTC (951024411)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -306,7 +306,7 @@
 EXT3-fs (mmcblk0p1): mounted filesystem with writeback data mode
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 308K (c0923000 - c0970000)
+Freeing unused kernel memory: 312K (c0924000 - c0972000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.759909] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.739966] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-09 13:37:08.942152156 -0700
+++ /tmp/foob	2015-08-09 13:37:08.942152156 -0700
@@ -42,18 +42,18 @@
 	 ##############################
 	 3.2 MiB/s
 done
-Bytes transferred = 6156392 (5df068 hex)
+Bytes transferred = 6156160 (5def80 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3.2 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 62999 (f617 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5df068 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5def80 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012616
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-27916-g74d3329 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Aug 2 19:03:13 PDT 2015
+Linux version 4.2.0-rc6-28106-gf7644cb (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Aug 9 13:33:03 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d0fa88   (11295 kB)
+      .text : 0xc0208000 - 0xc0d0faa0   (11295 kB)
       .init : 0xc0d10000 - 0xc0dfc000   ( 944 kB)
       .data : 0xc0dfc000 - 0xc0effb00   (1039 kB)
        .bss : 0xc0f02000 - 0xc0f53c28   ( 328 kB)
@@ -288,13 +288,13 @@
 +1.35V_LP0(sd2): supplied by +5V_SYS
 +1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
-tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 mmc1: MAN_BKOPS_EN bit is not set
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM16G 14.6 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 4.00 MiB
+tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
@@ -352,8 +352,8 @@
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
-EXT4-fs (mmcblk0p1): recovery complete
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
+EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-09 13:37:08.950152153 -0700
+++ /tmp/foob	2015-08-09 13:37:08.950152153 -0700
@@ -29,6 +29,7 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x81000000 zImage.tegra_defconfig
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.tegra_defconfig'.
@@ -41,18 +42,18 @@
 	 #####################
 	 3.2 MiB/s
 done
-Bytes transferred = 5067752 (4d53e8 hex)
+Bytes transferred = 5068016 (4d54f0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.9 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 62999 (f617 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4d53e8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4d54f0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012616
@@ -63,7 +64,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-27916-g74d3329 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Aug 2 19:03:46 PDT 2015
+Linux version 4.2.0-rc6-28106-gf7644cb (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Aug 9 13:33:28 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -71,7 +72,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09c4ec0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09c6ec0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -84,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002564K/2096128K available (6945K kernel code, 346K rwdata, 2376K rodata, 308K init, 363K bss, 28028K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002556K/2096128K available (6945K kernel code, 346K rwdata, 2380K rodata, 312K init, 363K bss, 28036K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,10 +93,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09228d0   (9323 kB)
-      .init : 0xc0923000 - 0xc0970000   ( 308 kB)
-      .data : 0xc0970000 - 0xc09c6a80   ( 347 kB)
-       .bss : 0xc09c9000 - 0xc0a23c58   ( 364 kB)
+      .text : 0xc0008000 - 0xc09238d4   (9327 kB)
+      .init : 0xc0924000 - 0xc0972000   ( 312 kB)
+      .data : 0xc0972000 - 0xc09c8a80   ( 347 kB)
+       .bss : 0xc09cb000 - 0xc0a25c58   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
 	Build-time adjustment of leaf fanout to 32.
@@ -339,7 +340,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:19 UTC (949363219)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:54 UTC (949363254)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 +2.8V_RUN_CAM_AF: disabling
@@ -359,7 +360,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 308K (c0923000 - c0970000)
+Freeing unused kernel memory: 312K (c0924000 - c0972000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-09 13:37:08.958152150 -0700
+++ /tmp/foob	2015-08-09 13:37:08.958152150 -0700
@@ -48,18 +48,18 @@
 	 ##############################
 	 4 MiB/s
 done
-Bytes transferred = 6156392 (5df068 hex)
+Bytes transferred = 6156160 (5def80 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5df068 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5def80 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-27916-g74d3329 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Aug 2 19:03:13 PDT 2015
+Linux version 4.2.0-rc6-28106-gf7644cb (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Aug 9 13:33:03 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -96,7 +96,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d0fa88   (11295 kB)
+      .text : 0xc0208000 - 0xc0d0faa0   (11295 kB)
       .init : 0xc0d10000 - 0xc0dfc000   ( 944 kB)
       .data : 0xc0dfc000 - 0xc0effb00   (1039 kB)
        .bss : 0xc0f02000 - 0xc0f53c28   ( 328 kB)
@@ -324,20 +324,18 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-mmc0: host does not support reading read-only switch, assuming write-enable
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:16:55 UTC (1419625015)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:23:19 UTC (1419625399)
+mmc0: host does not support reading read-only switch, assuming write-enable
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
@@ -346,7 +344,9 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 944K (c0d10000 - c0dfc000)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    2.337050] usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-09 13:37:08.962152148 -0700
+++ /tmp/foob	2015-08-09 13:37:08.966152146 -0700
@@ -47,7 +47,7 @@
 	 #####################
 	 4.1 MiB/s
 done
-Bytes transferred = 5067752 (4d53e8 hex)
+Bytes transferred = 5068016 (4d54f0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4d53e8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4d54f0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-27916-g74d3329 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Aug 2 19:03:46 PDT 2015
+Linux version 4.2.0-rc6-28106-gf7644cb (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Aug 9 13:33:28 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c09c4ec0, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c09c6ec0, node_mem_map eeff7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963468K/1048576K available (6945K kernel code, 346K rwdata, 2376K rodata, 308K init, 363K bss, 19572K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 963460K/1048576K available (6945K kernel code, 346K rwdata, 2380K rodata, 312K init, 363K bss, 19580K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,10 +99,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09228d0   (9323 kB)
-      .init : 0xc0923000 - 0xc0970000   ( 308 kB)
-      .data : 0xc0970000 - 0xc09c6a80   ( 347 kB)
-       .bss : 0xc09c9000 - 0xc0a23c58   ( 364 kB)
+      .text : 0xc0008000 - 0xc09238d4   (9327 kB)
+      .init : 0xc0924000 - 0xc0972000   ( 312 kB)
+      .data : 0xc0972000 - 0xc09c8a80   ( 347 kB)
+       .bss : 0xc09cb000 - 0xc0a25c58   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
 	Build-time adjustment of leaf fanout to 32.
@@ -345,12 +345,12 @@
 hub 1-1:1.0: 4 ports detected
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
- mmcblk1: p1 p2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:17:14 UTC (1419625034)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:23:37 UTC (1419625417)
 ALSA device list:
   #0: tegra-trimslice
 kjournald starting.  Commit interval 5 seconds
@@ -359,8 +359,8 @@
 EXT3-fs (mmcblk0p1): mounted filesystem with writeback data mode
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 308K (c0923000 - c0970000)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+Freeing unused kernel memory: 312K (c0924000 - c0972000)
+bash: cannot set[    2.235827] usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+ terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-09 13:37:08.970152145 -0700
+++ /tmp/foob	2015-08-09 13:37:08.974152144 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6156392 (5df068 hex)
+Bytes transferred = 6156160 (5def80 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5df068 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5def80 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-27916-g74d3329 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Aug 2 19:03:13 PDT 2015
+Linux version 4.2.0-rc6-28106-gf7644cb (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Aug 9 13:33:03 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -88,7 +88,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d0fa88   (11295 kB)
+      .text : 0xc0208000 - 0xc0d0faa0   (11295 kB)
       .init : 0xc0d10000 - 0xc0dfc000   ( 944 kB)
       .data : 0xc0dfc000 - 0xc0effb00   (1039 kB)
        .bss : 0xc0f02000 - 0xc0f53c28   ( 328 kB)
@@ -367,4 +367,4 @@
 Freeing unused kernel memory: 944K (c0d10000 - c0dfc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.763816] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.763778] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-09 13:37:08.978152142 -0700
+++ /tmp/foob	2015-08-09 13:37:08.982152139 -0700
@@ -39,18 +39,18 @@
 	 #####################
 	 3.3 MiB/s
 done
-Bytes transferred = 5067752 (4d53e8 hex)
+Bytes transferred = 5068016 (4d54f0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4d53e8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4d54f0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-27916-g74d3329 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Aug 2 19:03:46 PDT 2015
+Linux version 4.2.0-rc6-28106-gf7644cb (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Aug 9 13:33:28 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09c4ec0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09c6ec0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002588K/2096128K available (6945K kernel code, 346K rwdata, 2376K rodata, 308K init, 363K bss, 28004K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002580K/2096128K available (6945K kernel code, 346K rwdata, 2380K rodata, 312K init, 363K bss, 28012K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09228d0   (9323 kB)
-      .init : 0xc0923000 - 0xc0970000   ( 308 kB)
-      .data : 0xc0970000 - 0xc09c6a80   ( 347 kB)
-       .bss : 0xc09c9000 - 0xc0a23c58   ( 364 kB)
+      .text : 0xc0008000 - 0xc09238d4   (9327 kB)
+      .init : 0xc0924000 - 0xc0972000   ( 312 kB)
+      .data : 0xc0972000 - 0xc09c8a80   ( 347 kB)
+       .bss : 0xc09cb000 - 0xc0a25c58   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
 	Build-time adjustment of leaf fanout to 32.
@@ -366,7 +366,7 @@
 EXT3-fs (mmcblk0p1): mounted filesystem with writeback data mode
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 308K (c0923000 - c0970000)
+Freeing unused kernel memory: 312K (c0924000 - c0972000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.406366] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.416357] usb 2-1: new high-speed USB device number 2 using tegra-ehci
