Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-03-15 18:13:47.237723903 -0700
+++ /tmp/foob	2015-03-15 18:13:47.237723903 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.0.0-rc3+ (pwalmsley@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Mar 8 16:36:31 PDT 2015
+Linux version 4.0.0-rc4+ (pwalmsley@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Mar 15 18:06:04 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 Memory limited to 1024MB
@@ -17,7 +17,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 940360K/1048576K available (5013K kernel code, 341K rwdata, 1880K rodata, 256K init, 201K bss, 91832K reserved, 16384K cma-reserved)
+Memory: 940360K/1048576K available (5017K kernel code, 338K rwdata, 1880K rodata, 256K init, 200K bss, 91832K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -26,9 +26,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc00073e000 - 0xffffffc00077e000   (   256 KB)
-      .text : 0xffffffc000080000 - 0xffffffc00073d4e4   (  6902 KB)
-      .data : 0xffffffc000782000 - 0xffffffc0007d7600   (   342 KB)
+      .init : 0xffffffc00073f000 - 0xffffffc00077f000   (   256 KB)
+      .text : 0xffffffc000080000 - 0xffffffc00073e4e4   (  6906 KB)
+      .data : 0xffffffc000783000 - 0xffffffc0007d7800   (   338 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -83,7 +83,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.640:1): initialized
+audit: type=2000 audit(0.610:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.5.2
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
@@ -121,8 +121,8 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 256K (ffffffc00073e000 - ffffffc00077e000)
-Freeing alternatives memory: 8K (ffffffc00077e000 - ffffffc000780000)
+Freeing unused kernel memory: 256K (ffffffc00073f000 - ffffffc00077f000)
+Freeing alternatives memory: 8K (ffffffc00077f000 - ffffffc000781000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-03-15 18:13:47.253723901 -0700
+++ /tmp/foob	2015-03-15 18:13:47.253723901 -0700
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 #########################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6080664 (5cc898 hex)
+Bytes transferred = 6081544 (5ccc08 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 34638 (874e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5cc898 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5ccc08 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff74d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.0.0-rc3-24099-g9eccca0 (pwalmsley@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Mar 8 16:35:33 PDT 2015
+Linux version 4.0.0-rc4-24433-g06e5801 (pwalmsley@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Mar 15 18:05:06 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0ec7f00, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0ec7e40, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999648K/2096128K available (7830K kernel code, 1015K rwdata, 3412K rodata, 800K init, 314K bss, 30944K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999648K/2096128K available (7830K kernel code, 1015K rwdata, 3412K rodata, 800K init, 316K bss, 30944K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,14 +88,14 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d03b28   (11247 kB)
+      .text : 0xc0208000 - 0xc0d03b54   (11247 kB)
       .init : 0xc0d04000 - 0xc0dcc000   ( 800 kB)
-      .data : 0xc0dcc000 - 0xc0ec9e20   (1016 kB)
-       .bss : 0xc0ec9e20 - 0xc0f18968   ( 315 kB)
+      .data : 0xc0dcc000 - 0xc0ec9d60   (1016 kB)
+       .bss : 0xc0ec9d60 - 0xc0f18e68   ( 317 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
-	RCU restricting CPUs from NR_CPUS=8 to nr_cpu_ids=4.
+	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
 L2C: failed to init: -19
@@ -115,7 +115,7 @@
 /cpus/cpu@2 missing clock-frequency property
 /cpus/cpu@3 missing clock-frequency property
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
-Setting up static identity map for 0x80967690 - 0x80967728
+Setting up static identity map for 0x80967a60 - 0x80967af8
 Tegra Revision: A01 SKU: 0 CPU Process: 1 Core Process: 0
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
 CPU2: thread -1, cpu 2, socket 0, mpidr 80000002
@@ -169,7 +169,7 @@
 vdd-com-3v3: supplied by vdd-sys-3v3
 vdd-sby-5v0: supplied by vdd_ac_bat
 vdd-sby-3v3: supplied by vdd_ac_bat
-genirq: Setting trigger mode 8 for irq 341 failed (gic_set_type+0x0/0x9c)
+genirq: Setting trigger mode 8 for irq 341 failed (gic_set_type+0x0/0xa4)
 palmas 2-0058: Irq flag is 0x00000000
 palmas 2-0058: Muxing GPIO ff, PWM 0, LED 0
 vddio-ddr: supplied by vdd-ao
@@ -334,7 +334,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-08-13 18:57:33 UTC (966193053)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-08-20 20:27:25 UTC (966803245)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -348,4 +348,4 @@
 Freeing unused kernel memory: 800K (c0d04000 - c0dcc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.778744] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.793924] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-03-15 18:13:47.261723900 -0700
+++ /tmp/foob	2015-03-15 18:13:47.261723900 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ####
-	 3.1 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 4827192 (49a838 hex)
+Bytes transferred = 4827552 (49a9a0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34638 (874e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x49a838 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x49a9a0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff74d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.0.0-rc3-129315-g9eccca0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Mar 8 16:34:06 PDT 2015
+Linux version 4.0.0-rc4-129649-g06e5801 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Mar 15 18:03:44 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0943c40, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c0943c80, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc08aac70   (8844 kB)
+      .text : 0xc0008000 - 0xc08aac90   (8844 kB)
       .init : 0xc08ab000 - 0xc08f6000   ( 300 kB)
-      .data : 0xc08f6000 - 0xc0945720   ( 318 kB)
-       .bss : 0xc0945720 - 0xc099d370   ( 352 kB)
+      .data : 0xc08f6000 - 0xc0945760   ( 318 kB)
+       .bss : 0xc0945760 - 0xc099d3b0   ( 352 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
 NR_IRQS:16 nr_irqs:16 16
@@ -116,7 +116,7 @@
 /cpus/cpu@2 missing clock-frequency property
 /cpus/cpu@3 missing clock-frequency property
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
-Setting up static identity map for 0x806441b8 - 0x80644250
+Setting up static identity map for 0x806444e0 - 0x80644578
 Tegra Revision: A01 SKU: 0 CPU Process: 1 Core Process: 0
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
 CPU2: thread -1, cpu 2, socket 0, mpidr 80000002
@@ -169,7 +169,7 @@
 vdd-com-3v3: supplied by vdd-sys-3v3
 vdd-sby-5v0: supplied by vdd_ac_bat
 vdd-sby-3v3: supplied by vdd_ac_bat
-genirq: Setting trigger mode 8 for irq 341 failed (gic_set_type+0x0/0x94)
+genirq: Setting trigger mode 8 for irq 341 failed (gic_set_type+0x0/0x9c)
 palmas 2-0058: Irq flag is 0x00000000
 palmas 2-0058: Muxing GPIO ff, PWM 0, LED 0
 vddio-ddr: supplied by vdd-ao
@@ -325,7 +325,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-08-13 18:56:37 UTC (966192997)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-08-20 20:26:30 UTC (966803190)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -340,4 +340,4 @@
 Freeing unused kernel memory: 300K (c08ab000 - c08f6000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.939769] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.939897] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-03-15 18:13:47.269723899 -0700
+++ /tmp/foob	2015-03-15 18:13:47.269723899 -0700
@@ -40,9 +40,9 @@
 	 #################################################################
 	 #################################################################
 	 #########################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6080664 (5cc898 hex)
+Bytes transferred = 6081544 (5ccc08 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -53,7 +53,7 @@
 done
 Bytes transferred = 49059 (bfa3 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5cc898 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5ccc08 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 8200efa2
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.0.0-rc3-24099-g9eccca0 (pwalmsley@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Mar 8 16:35:33 PDT 2015
+Linux version 4.0.0-rc4-24433-g06e5801 (pwalmsley@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Mar 15 18:05:06 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0ec7f00, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0ec7e40, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999580K/2096128K available (7830K kernel code, 1015K rwdata, 3412K rodata, 800K init, 314K bss, 31012K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999580K/2096128K available (7830K kernel code, 1015K rwdata, 3412K rodata, 800K init, 316K bss, 31012K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,14 +90,14 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d03b28   (11247 kB)
+      .text : 0xc0208000 - 0xc0d03b54   (11247 kB)
       .init : 0xc0d04000 - 0xc0dcc000   ( 800 kB)
-      .data : 0xc0dcc000 - 0xc0ec9e20   (1016 kB)
-       .bss : 0xc0ec9e20 - 0xc0f18968   ( 315 kB)
+      .data : 0xc0dcc000 - 0xc0ec9d60   (1016 kB)
+       .bss : 0xc0ec9d60 - 0xc0f18e68   ( 317 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
-	RCU restricting CPUs from NR_CPUS=8 to nr_cpu_ids=4.
+	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
 L2C: failed to init: -19
@@ -117,7 +117,7 @@
 /cpus/cpu@2 missing clock-frequency property
 /cpus/cpu@3 missing clock-frequency property
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
-Setting up static identity map for 0x80967690 - 0x80967728
+Setting up static identity map for 0x80967a60 - 0x80967af8
 Tegra Revision: A01 SKU: 129 CPU Process: 1 Core Process: 1
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
 CPU2: thread -1, cpu 2, socket 0, mpidr 80000002
@@ -399,7 +399,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:17 UTC (949363217)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:18 UTC (949363218)
 +1.8V_RUN_VPP_FUSE: disabling
 +2.8V_RUN_CAM_AF: disabling
 +3.3V_RUN_TOUCH: disabling
@@ -419,4 +419,4 @@
 Freeing unused kernel memory: 800K (c0d04000 - c0dcc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    5.735444] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    5.740799] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-03-15 18:13:47.277723897 -0700
+++ /tmp/foob	2015-03-15 18:13:47.281723897 -0700
@@ -41,18 +41,18 @@
 	 ####
 	 3.2 MiB/s
 done
-Bytes transferred = 4827192 (49a838 hex)
+Bytes transferred = 4827552 (49a9a0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 49059 (bfa3 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x49a838 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x49a9a0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 8200efa2
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.0.0-rc3-129315-g9eccca0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Mar 8 16:34:06 PDT 2015
+Linux version 4.0.0-rc4-129649-g06e5801 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Mar 15 18:03:44 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0943c40, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c0943c80, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc08aac70   (8844 kB)
+      .text : 0xc0008000 - 0xc08aac90   (8844 kB)
       .init : 0xc08ab000 - 0xc08f6000   ( 300 kB)
-      .data : 0xc08f6000 - 0xc0945720   ( 318 kB)
-       .bss : 0xc0945720 - 0xc099d370   ( 352 kB)
+      .data : 0xc08f6000 - 0xc0945760   ( 318 kB)
+       .bss : 0xc0945760 - 0xc099d3b0   ( 352 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
 NR_IRQS:16 nr_irqs:16 16
@@ -118,7 +118,7 @@
 /cpus/cpu@2 missing clock-frequency property
 /cpus/cpu@3 missing clock-frequency property
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
-Setting up static identity map for 0x806441b8 - 0x80644250
+Setting up static identity map for 0x806444e0 - 0x80644578
 Tegra Revision: A01 SKU: 129 CPU Process: 1 Core Process: 1
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
 CPU2: thread -1, cpu 2, socket 0, mpidr 80000002
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-03-15 18:13:47.285723897 -0700
+++ /tmp/foob	2015-03-15 18:13:47.289723896 -0700
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 #########################
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 6080664 (5cc898 hex)
+Bytes transferred = 6081544 (5ccc08 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 24641 (6041 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5cc898 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5ccc08 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff6000, end 8ffff040 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.0.0-rc3-24099-g9eccca0 (pwalmsley@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Mar 8 16:35:33 PDT 2015
+Linux version 4.0.0-rc4-24433-g06e5801 (pwalmsley@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Mar 15 18:05:06 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0ec7f00, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0ec7e40, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999696K/2096128K available (7830K kernel code, 1015K rwdata, 3412K rodata, 800K init, 314K bss, 30896K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999696K/2096128K available (7830K kernel code, 1015K rwdata, 3412K rodata, 800K init, 316K bss, 30896K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,14 +88,14 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d03b28   (11247 kB)
+      .text : 0xc0208000 - 0xc0d03b54   (11247 kB)
       .init : 0xc0d04000 - 0xc0dcc000   ( 800 kB)
-      .data : 0xc0dcc000 - 0xc0ec9e20   (1016 kB)
-       .bss : 0xc0ec9e20 - 0xc0f18968   ( 315 kB)
+      .data : 0xc0dcc000 - 0xc0ec9d60   (1016 kB)
+       .bss : 0xc0ec9d60 - 0xc0f18e68   ( 317 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
-	RCU restricting CPUs from NR_CPUS=8 to nr_cpu_ids=4.
+	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
 L2C: platform modifies aux control register: 0x02080000 -> 0x3e480001
@@ -116,7 +116,7 @@
 Mountpoint-cache hash table entries: 2048 (order: 1, 8192 bytes)
 CPU: Testing write buffer coherency: ok
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
-Setting up static identity map for 0x80967690 - 0x80967728
+Setting up static identity map for 0x80967a60 - 0x80967af8
 Tegra Revision: A03 SKU: 128 CPU Process: 4 Core Process: 0
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
 CPU2: thread -1, cpu 2, socket 0, mpidr 80000002
@@ -387,4 +387,4 @@
 Freeing unused kernel memory: 800K (c0d04000 - c0dcc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.203430] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.203473] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-03-15 18:13:47.321723893 -0700
+++ /tmp/foob	2015-03-15 18:13:47.321723893 -0700
@@ -39,7 +39,7 @@
 	 ####
 	 3.4 MiB/s
 done
-Bytes transferred = 4827192 (49a838 hex)
+Bytes transferred = 4827552 (49a9a0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 24641 (6041 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x49a838 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x49a9a0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff6000, end 8ffff040 ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.0.0-rc3-129315-g9eccca0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Mar 8 16:34:06 PDT 2015
+Linux version 4.0.0-rc4-129649-g06e5801 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Mar 15 18:03:44 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0943c40, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c0943c80, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc08aac70   (8844 kB)
+      .text : 0xc0008000 - 0xc08aac90   (8844 kB)
       .init : 0xc08ab000 - 0xc08f6000   ( 300 kB)
-      .data : 0xc08f6000 - 0xc0945720   ( 318 kB)
-       .bss : 0xc0945720 - 0xc099d370   ( 352 kB)
+      .data : 0xc08f6000 - 0xc0945760   ( 318 kB)
+       .bss : 0xc0945760 - 0xc099d3b0   ( 352 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
 NR_IRQS:16 nr_irqs:16 16
@@ -117,7 +117,7 @@
 Initializing cgroup subsys debug
 CPU: Testing write buffer coherency: ok
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
-Setting up static identity map for 0x806441b8 - 0x80644250
+Setting up static identity map for 0x806444e0 - 0x80644578
 Tegra Revision: A03 SKU: 128 CPU Process: 4 Core Process: 0
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
 CPU2: thread -1, cpu 2, socket 0, mpidr 80000002
@@ -379,4 +379,4 @@
 Freeing unused kernel memory: 300K (c08ab000 - c08f6000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.476417] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.466457] usb 2-1: new high-speed USB device number 2 using tegra-ehci
