Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-09-16 16:45:51.803229565 -0700
+++ /tmp/foob	2017-09-16 16:45:51.807229583 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-187049-g569dbb8 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Sep 3 14:10:21 PDT 2017
+Linux version 4.14.0-rc1-199401-g2bd6bf0 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sat Sep 16 16:42:19 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
@@ -8,7 +8,7 @@
 cma: Reserved 16 MiB at 0x000000007f000000
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000007fffffff]
-NUMA: NODE_DATA [mem 0x7efe8d80-0x7efea87f]
+NUMA: NODE_DATA [mem 0x7efeaf00-0x7efec9ff]
 Zone ranges:
   DMA      [mem 0x0000000040000000-0x000000007fffffff]
   Normal   empty
@@ -20,23 +20,24 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: Trusted OS migration not required
-percpu: Embedded 26 pages/cpu @ffff80003efcb000 s67456 r8192 d30848 u106496
+random: fast init done
+percpu: Embedded 23 pages/cpu @ffff80003efd0000 s55552 r8192 d30464 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
-Built 1 zonelists in Node order, mobility grouping on.  Total pages: 258048
+Built 1 zonelists, mobility grouping on.  Total pages: 258048
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 999292K/1048576K available (9084K kernel code, 1128K rwdata, 4448K rodata, 1088K init, 403K bss, 32900K reserved, 16384K cma-reserved)
+Memory: 999040K/1048576K available (9212K kernel code, 1142K rwdata, 4568K rodata, 1088K init, 402K bss, 33152K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008960000   (  9088 KB)
-    .rodata : 0xffff000008960000 - 0xffff000008dc0000   (  4480 KB)
-      .init : 0xffff000008dc0000 - 0xffff000008ed0000   (  1088 KB)
-      .data : 0xffff000008ed0000 - 0xffff000008fea200   (  1129 KB)
-       .bss : 0xffff000008fea200 - 0xffff00000904f070   (   404 KB)
+      .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
+    .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
+      .init : 0xffff000008e00000 - 0xffff000008f10000   (  1088 KB)
+      .data : 0xffff000008f10000 - 0xffff00000902da00   (  1143 KB)
+       .bss : 0xffff00000902da00 - 0xffff000009092470   (   403 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -45,6 +46,7 @@
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=1.
+	Tasks RCU enabled.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=1
 NR_IRQS: 64, nr_irqs: 64, preallocated irqs: 0
 GICv2m: range[mem 0x08020000-0x08020fff], SPI[80:143]
@@ -72,11 +74,10 @@
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
 futex hash table entries: 256 (order: 3, 32768 bytes)
 pinctrl core: initialized pinctrl subsystem
-random: fast init done
 DMI not present or invalid.
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008966000, 1 data @ ffff000008ed5000)
+vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f15000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -113,7 +114,7 @@
 hw perfevents: enabled with armv8_pmuv3 PMU driver, 1 counters available
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.767:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.940:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -121,7 +122,7 @@
 Key type id_legacy registered
 nfs4filelayout_init: NFSv4 File Layout Driver Registering...
 9p: Installing v9fs 9p2000 file system support
-Block layer SCSI generic (bsg) driver version 0.4 loaded (major 247)
+Block layer SCSI generic (bsg) driver version 0.4 loaded (major 246)
 io scheduler noop registered
 io scheduler cfq registered (default)
 io scheduler mq-deadline registered
@@ -143,7 +144,6 @@
 cacheinfo: Unable to detect cache hierarchy for CPU 0
 loop: module loaded
  vda: vda1 vda2
-hisi_sas: driver version v1.6
 libphy: Fixed MDIO Bus: probed
 tun: Universal TUN/TAP device driver, 1.6
 e1000e: Intel(R) PRO/1000 Network Driver - 3.2.6-k
@@ -179,7 +179,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-09-03 21:10:57 UTC (1504473057)
+rtc-pl031 9010000.pl031: setting system clock to 2017-09-16 23:43:01 UTC (1505605381)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-09-16 16:45:51.835229699 -0700
+++ /tmp/foob	2017-09-16 16:45:51.839229715 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800962 us
+Bootloader-Cpu Init at (time stamp): 1801888 us
 
 Pinmux changes applied in kernel way
 
@@ -59,25 +59,25 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 4 us
+Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800935  us
-Bootloader (AosInit) time: 734259 us
-Bootloader Main Init time: 180089 us
-BlockDevMgrInit time: 157519 us
+Miscellaneous avp time: 1801862  us
+Bootloader (AosInit) time: 734248 us
+Bootloader Main Init time: 180084 us
+BlockDevMgrInit time: 157536 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
-FsMgrInit time: 2 us
-Partition Table load: 76692 us 
-Display init time: 574796 us
-Kernel read time: 1319 msec
-Load Kernel Size:   17483776 Bytes
-Bl Kernel ReadSpeed: 13255000 Bytes/sec
-Pre-Kernel Jump time: 3775957 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 8716097 us
+StorMgrInit time: 30 us
+FsMgrInit time: 3 us
+Partition Table load: 76681 us 
+Display init time: 574762 us
+Kernel read time: 1339 msec
+Load Kernel Size:   17760256 Bytes
+Bl Kernel ReadSpeed: 13263000 Bytes/sec
+Pre-Kernel Jump time: 3827558 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 8788706 us
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-187049-g569dbb8 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Sep 3 14:10:21 PDT 2017
+Linux version 4.14.0-rc1-199401-g2bd6bf0 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sat Sep 16 16:42:19 PDT 2017
 Boot CPU: AArch64 Processor [4e0f0000]
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
@@ -86,7 +86,7 @@
 cma: Reserved 16 MiB at 0x00000000eb400000
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x00000000ec5fffff]
-NUMA: NODE_DATA [mem 0xec5be700-0xec5c01ff]
+NUMA: NODE_DATA [mem 0xec5c0d80-0xec5c287f]
 Zone ranges:
   DMA      [mem 0x0000000080000000-0x00000000ec5fffff]
   Normal   empty
@@ -95,21 +95,22 @@
   node   0: [mem 0x0000000080000000-0x00000000ec5fffff]
 Initmem setup node 0 [mem 0x0000000080000000-0x00000000ec5fffff]
 /cpus/cpu@1: missing enable-method property
-percpu: Embedded 26 pages/cpu @ffff80006c59f000 s67456 r8192 d30848 u106496
+random: fast init done
+percpu: Embedded 23 pages/cpu @ffff80006c5a4000 s55552 r8192 d30464 u94208
 Detected PIPT I-cache on CPU0
-Built 1 zonelists in Node order, mobility grouping on.  Total pages: 436968
+Built 1 zonelists, mobility grouping on.  Total pages: 436968
 Policy zone: DMA
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=192511 modem_id=0 android.kerneltype=normal
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 1708512K/1775616K available (9084K kernel code, 1128K rwdata, 4448K rodata, 1088K init, 403K bss, 50720K reserved, 16384K cma-reserved)
+Memory: 1708264K/1775616K available (9212K kernel code, 1142K rwdata, 4568K rodata, 1088K init, 402K bss, 50968K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008960000   (  9088 KB)
-    .rodata : 0xffff000008960000 - 0xffff000008dc0000   (  4480 KB)
-      .init : 0xffff000008dc0000 - 0xffff000008ed0000   (  1088 KB)
-      .data : 0xffff000008ed0000 - 0xffff000008fea200   (  1129 KB)
-       .bss : 0xffff000008fea200 - 0xffff00000904f070   (   404 KB)
+      .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
+    .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
+      .init : 0xffff000008e00000 - 0xffff000008f10000   (  1088 KB)
+      .data : 0xffff000008f10000 - 0xffff00000902da00   (  1143 KB)
+       .bss : 0xffff00000902da00 - 0xffff000009092470   (   403 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -118,6 +119,7 @@
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=1.
+	Tasks RCU enabled.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=1
 NR_IRQS: 64, nr_irqs: 64, preallocated irqs: 0
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
@@ -145,11 +147,10 @@
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
 futex hash table entries: 256 (order: 3, 32768 bytes)
 pinctrl core: initialized pinctrl subsystem
-random: fast init done
 DMI not present or invalid.
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008966000, 1 data @ ffff000008ed5000)
+vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f15000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -199,7 +200,7 @@
 Freeing initrd memory: 1284K
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.929:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.975:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -207,7 +208,7 @@
 Key type id_legacy registered
 nfs4filelayout_init: NFSv4 File Layout Driver Registering...
 9p: Installing v9fs 9p2000 file system support
-Block layer SCSI generic (bsg) driver version 0.4 loaded (major 247)
+Block layer SCSI generic (bsg) driver version 0.4 loaded (major 246)
 io scheduler noop registered
 io scheduler cfq registered (default)
 io scheduler mq-deadline registered
@@ -223,7 +224,6 @@
 msm_serial: driver initialized
 cacheinfo: Unable to detect cache hierarchy for CPU 0
 loop: module loaded
-hisi_sas: driver version v1.6
 libphy: Fixed MDIO Bus: probed
 tun: Universal TUN/TAP device driver, 1.6
 e1000e: Intel(R) PRO/1000 Network Driver - 3.2.6-k
@@ -279,6 +279,7 @@
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
 hctosys: unable to open rtc device (rtc0)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
@@ -289,7 +290,6 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 1088K
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-09-16 16:45:51.863229814 -0700
+++ /tmp/foob	2017-09-16 16:45:51.867229832 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 769174 us
-Basic modules init took 350470 us
+Tegraboot started after 755866 us
+Basic modules init took 348743 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14365 us
-Carveout took 252737 us
-CPU initialization took 92432 us
-Total time taken by TegraBoot 710004 us
+Next stage binary read took 14370 us
+Carveout took 252696 us
+CPU initialization took 92435 us
+Total time taken by TegraBoot 708244 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.609] LPDDR4 Training: Number of tables = 10
+64b[0001.594] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16163328 bytes read in 822 ms (18.8 MiB/s)
+16439808 bytes read in 835 ms (18.8 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-45170 bytes read in 460 ms (95.7 KiB/s)
+45170 bytes read in 455 ms (96.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-187049-g569dbb8 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Sep 3 14:10:21 PDT 2017
+Linux version 4.14.0-rc1-199401-g2bd6bf0 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sat Sep 16 16:42:19 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
@@ -142,7 +142,7 @@
 cma: Reserved 16 MiB at 0x00000000fe000000
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
-NUMA: NODE_DATA [mem 0x13ffab080-0x13ffacb7f]
+NUMA: NODE_DATA [mem 0x13ffada80-0x13ffaf57f]
 Zone ranges:
   DMA      [mem 0x0000000080000000-0x00000000ffffffff]
   Normal   [mem 0x0000000100000000-0x000000013fffffff]
@@ -161,26 +161,27 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_TYPE not supported.
-percpu: Embedded 26 pages/cpu @ffff8000bff40000 s67456 r8192 d30848 u106496
-pcpu-alloc: s67456 r8192 d30848 u106496 alloc=26*4096
+random: fast init done
+percpu: Embedded 23 pages/cpu @ffff8000bff4e000 s55552 r8192 d30464 u94208
+pcpu-alloc: s55552 r8192 d30464 u94208 alloc=23*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
-Built 1 zonelists in Node order, mobility grouping on.  Total pages: 770560
+Built 1 zonelists, mobility grouping on.  Total pages: 770560
 Policy zone: Normal
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2982844K/3131392K available (9084K kernel code, 1128K rwdata, 4448K rodata, 1088K init, 403K bss, 132164K reserved, 16384K cma-reserved)
+Memory: 2982628K/3131392K available (9212K kernel code, 1142K rwdata, 4568K rodata, 1088K init, 402K bss, 132380K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008960000   (  9088 KB)
-    .rodata : 0xffff000008960000 - 0xffff000008dc0000   (  4480 KB)
-      .init : 0xffff000008dc0000 - 0xffff000008ed0000   (  1088 KB)
-      .data : 0xffff000008ed0000 - 0xffff000008fea200   (  1129 KB)
-       .bss : 0xffff000008fea200 - 0xffff00000904f070   (   404 KB)
+      .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
+    .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
+      .init : 0xffff000008e00000 - 0xffff000008f10000   (  1088 KB)
+      .data : 0xffff000008f10000 - 0xffff00000902da00   (  1143 KB)
+       .bss : 0xffff00000902da00 - 0xffff000009092470   (   403 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -189,6 +190,7 @@
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
+	Tasks RCU enabled.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS: 64, nr_irqs: 64, preallocated irqs: 0
 GIC: Using split EOI/Deactivate mode
@@ -228,11 +230,10 @@
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 pinctrl core: initialized pinctrl subsystem
-random: fast init done
 DMI not present or invalid.
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008966000, 1 data @ ffff000008ed5000)
+vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f15000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -267,14 +268,14 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80950000
+kvm [1]: IDMAP page: 80973000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.523:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.519:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -282,7 +283,7 @@
 Key type id_legacy registered
 nfs4filelayout_init: NFSv4 File Layout Driver Registering...
 9p: Installing v9fs 9p2000 file system support
-Block layer SCSI generic (bsg) driver version 0.4 loaded (major 247)
+Block layer SCSI generic (bsg) driver version 0.4 loaded (major 246)
 io scheduler noop registered
 io scheduler cfq registered (default)
 io scheduler mq-deadline registered
@@ -300,7 +301,6 @@
 msm_serial: driver initialized
 cacheinfo: Unable to detect cache hierarchy for CPU 0
 loop: module loaded
-hisi_sas: driver version v1.6
 libphy: Fixed MDIO Bus: probed
 tun: Universal TUN/TAP device driver, 1.6
 e1000e: Intel(R) PRO/1000 Network Driver - 3.2.6-k
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-09-16 16:45:51.875229867 -0700
+++ /tmp/foob	2017-09-16 16:45:51.875229867 -0700
@@ -1,74 +1,5 @@
-t0] enabled
-NUMA: No NUMA configuration found
-NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
-NUMA: NODE_DATA [mem 0x13ff9f100-0x13ffa0bff]
-Zone ranges:
-  DMA      [mem 0x0000000080000000-0x00000000ffffffff]
-  Normal   [mem 0x0000000100000000-0x000000013fffffff]
-Movable zone start for each node
-Early memory node ranges
-  node   0: [mem 0x0000000080000000-0x00000000f65fffff]
-  node   0: [mem 0x0000000100000000-0x000000013fffffff]
-Initmem setup node 0 [mem 0x0000000080000000-0x000000013fffffff]
-psci: probing for conduit method from DT.
-psci: PSCIv1.0 detected in firmware.
-psci: Using standard PSCI v0.2 function IDs
-psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 26 pages/cpu @ffff8000bff34000 s67456 r8192 d30848 u106496
-Detected PIPT I-cache on CPU0
-CPU features: enabling workaround for ARM erratum 832075
-CPU features: enabling workaround for ARM erratum 834220
-Built 1 zonelists in Node order, mobility grouping on.  Total pages: 735336
-Policy zone: Normal
-Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
-PID hash table entries: 4096 (order: 3, 32768 bytes)
-software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2838112K/2988032K available (9084K kernel code, 1128K rwdata, 4448K rodata, 1088K init, 403K bss, 133536K reserved, 16384K cma-reserved)
-Virtual kernel memory layout:
-    modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
-    vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008960000   (  9088 KB)
-    .rodata : 0xffff000008960000 - 0xffff000008dc0000   (  4480 KB)
-      .init : 0xffff000008dc0000 - 0xffff000008ed0000   (  1088 KB)
-      .data : 0xffff000008ed0000 - 0xffff000008fea200   (  1129 KB)
-       .bss : 0xffff000008fea200 - 0xffff00000904f070   (   404 KB)
-    fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
-    PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
-    vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
-              0xffff7e0000000000 - 0xffff7e0003000000   (    48 MB actual)
-    memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
-SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
-Preemptible hierarchical RCU implementation.
-	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
-NR_IRQS: 64, nr_irqs: 64, preallocated irqs: 0
-GIC: Using split EOI/Deactivate mode
-/interrupt-controller@60004000: 192 interrupts forwarded to /interrupt-controller@50041000
-boot misc0 0x0: expected 0x80000
- (comparison mask = 0x400ffffb)
-boot misc2 0x1f720005: expected 0x1f720f05
- (comparison mask = 0xffffff17)
-PLL_C already enabled. Postponing set full defaults
-PLL_DP already enabled. Postponing set full defaults
-boot misc0 0x0: expected 0x20
- (comparison mask = 0x7ffffff)
-boot misc1 0x90000000: expected 0xc0000000
- (comparison mask = 0xf8000000)
-boot misc2 0x0: expected 0xf400f0da
- (comparison mask = 0xffffffff)
-boot misc3 0x8ce: expected 0x2004f400
- (comparison mask = 0xffffffff)
-boot misc1 0x0: expected 0x20
- (comparison mask = 0xffffff)
-PLL_X already enabled. Postponing set full defaults
-arch_timer: cp15 timer(s) running at 19.20MHz (phys).
-clocksource: arch_sys_counter: mask: 0xffffffffffffff max_cycles: 0x46d987e47, max_idle_ns: 440795202767 ns
-sched_clock: 56 bits at 19MHz, resolution 52ns, wraps every 4398046511078ns
-Console: colour dummy device 80x25
-console [tty0] enabled
-bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-187049-g569dbb8 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Sep 3 14:10:21 PDT 2017
+Linux version 4.14.0-rc1-199401-g2bd6bf0 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sat Sep 16 16:42:19 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
@@ -78,7 +9,7 @@
 bootconsole [uart0] enabled
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
-NUMA: NODE_DATA [mem 0x13ff9f100-0x13ffa0bff]
+NUMA: NODE_DATA [mem 0x13ffa1c80-0x13ffa377f]
 Zone ranges:
   DMA      [mem 0x0000000080000000-0x00000000ffffffff]
   Normal   [mem 0x0000000100000000-0x000000013fffffff]
@@ -91,24 +22,24 @@
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 26 pages/cpu @ffff8000bff34000 s67456 r8192 d30848 u106496
+percpu: Embedded 23 pages/cpu @ffff8000bff42000 s55552 r8192 d30464 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
-Built 1 zonelists in Node order, mobility grouping on.  Total pages: 735336
+Built 1 zonelists, mobility grouping on.  Total pages: 735336
 Policy zone: Normal
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2838112K/2988032K available (9084K kernel code, 1128K rwdata, 4448K rodata, 1088K init, 403K bss, 133536K reserved, 16384K cma-reserved)
+Memory: 2837892K/2988032K available (9212K kernel code, 1142K rwdata, 4568K rodata, 1088K init, 402K bss, 133756K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008960000   (  9088 KB)
-    .rodata : 0xffff000008960000 - 0xffff000008dc0000   (  4480 KB)
-      .init : 0xffff000008dc0000 - 0xffff000008ed0000   (  1088 KB)
-      .data : 0xffff000008ed0000 - 0xffff000008fea200   (  1129 KB)
-       .bss : 0xffff000008fea200 - 0xffff00000904f070   (   404 KB)
+      .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
+    .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
+      .init : 0xffff000008e00000 - 0xffff000008f10000   (  1088 KB)
+      .data : 0xffff000008f10000 - 0xffff00000902da00   (  1143 KB)
+       .bss : 0xffff00000902da00 - 0xffff000009092470   (   403 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -117,6 +48,7 @@
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
+	Tasks RCU enabled.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS: 64, nr_irqs: 64, preallocated irqs: 0
 GIC: Using split EOI/Deactivate mode
@@ -126,15 +58,15 @@
 boot misc2 0x1f720005: expected 0x1f720f05
  (comparison mask = 0xffffff17)
 PLL_C already enabled. Postponing set full defaults
-PLL_DP already enabled. Postponing set full defaults
 boot misc0 0x0: expected 0x20
  (comparison mask = 0x7ffffff)
-boot misc1 0x90000000: expected 0xc0000000
+boot misc1 0x10000000: expected 0xc0000000
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x8ce: expected 0x2004f400
+boot misc3 0x0: expected 0x2004f400
  (comparison mask = 0xffffffff)
+PLL_DP already enabled. Postponing set full defaults
 boot misc1 0x0: expected 0x20
  (comparison mask = 0xffffff)
 PLL_X already enabled. Postponing set full defaults
@@ -154,6 +86,7 @@
 ASID allocator initialised with 65536 entries
 Hierarchical SRCU implementation.
 Speedo Revision 4
+random: fast init done
 Tegra Revision: A02 SKU: 19 CPU Process: 0 SoC Process: 0
 EFI services will not be available.
 smp: Bringing up secondary CPUs ...
@@ -172,11 +105,10 @@
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 pinctrl core: initialized pinctrl subsystem
-random: fast init done
 DMI not present or invalid.
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008966000, 1 data @ ffff000008ed5000)
+vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f15000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -216,16 +148,16 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 1280K
+Freeing initrd memory: 1284K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80950000
+kvm [1]: IDMAP page: 80973000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.270:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.251:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -233,7 +165,7 @@
 Key type id_legacy registered
 nfs4filelayout_init: NFSv4 File Layout Driver Registering...
 9p: Installing v9fs 9p2000 file system support
-Block layer SCSI generic (bsg) driver version 0.4 loaded (major 247)
+Block layer SCSI generic (bsg) driver version 0.4 loaded (major 246)
 io scheduler noop registered
 io scheduler cfq registered (default)
 io scheduler mq-deadline registered
@@ -268,7 +200,6 @@
 PP_LDO6: Bringing 2800000uV into 1800000-1800000uV
 max77620-ldo7: supplied by regulator-dummy
 max77620-ldo8: supplied by regulator-dummy
-hisi_sas: driver version v1.6
 libphy: Fixed MDIO Bus: probed
 tun: Universal TUN/TAP device driver, 1.6
 e1000e: Intel(R) PRO/1000 Network Driver - 3.2.6-k
@@ -307,7 +238,6 @@
 tegra_rtc 7000e000.rtc: Tegra internal Real Time Clock
 i2c /dev entries driver
 (NULL device *): hwmon_device_register() is deprecated. Please convert the driver to use hwmon_device_register_with_info().
-bq27xxx-battery 2-0055: support ver. 1.2.0 enabled
 sdhci: Secure Digital Host Controller Interface driver
 sdhci: Copyright(c) Pierre Ossman
 Synopsys Designware Multimedia Card Interface Driver
@@ -330,7 +260,7 @@
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2001-06-15 02:34:36 UTC (992572476)
+max77686-rtc max77620-rtc: setting system clock to 2001-06-28 05:06:29 UTC (993704789)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -341,8 +271,8 @@
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
-Freeing unused kernel memory: 1088K
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+Freeing unused kernel memory: 1088K
 
 System initialization...
 
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
