Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-10-08 21:44:31.488559316 -0700
+++ /tmp/foob	2017-10-08 21:44:31.492559333 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc3-200087-g9e66317 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Oct 1 15:39:11 PDT 2017
+Linux version 4.14.0-rc4-318799-g8a5776a5f498 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Oct 8 21:42:18 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
@@ -114,7 +114,7 @@
 hw perfevents: enabled with armv8_pmuv3 PMU driver, 1 counters available
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.996:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.828:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -179,7 +179,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-10-01 22:40:11 UTC (1506897611)
+rtc-pl031 9010000.pl031: setting system clock to 2017-10-09 04:43:25 UTC (1507524205)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-10-08 21:44:31.556559610 -0700
+++ /tmp/foob	2017-10-08 21:44:31.556559610 -0700
@@ -35,7 +35,7 @@
 	 #################################################################
 	 #################################################################
 	 ####################################
-	 3.3 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 8159752 (7c8208 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc3-200087-g9e66317 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Oct 1 15:36:12 PDT 2017
+Linux version 4.14.0-rc4-200558-g8a5776a (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Oct 8 21:37:02 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993356K/2096128K available (10240K kernel code, 1631K rwdata, 4500K rodata, 2048K init, 405K bss, 37236K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993356K/2096128K available (10240K kernel code, 1631K rwdata, 4504K rodata, 2048K init, 405K bss, 37236K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -467,8 +467,8 @@
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
 pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
-ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
+ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 pci 0000:00:02.0: enabling Extended Tags
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x88 may corrupt adjacent RW1C bits
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-10-08 21:44:31.572559679 -0700
+++ /tmp/foob	2017-10-08 21:44:31.576559697 -0700
@@ -33,20 +33,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3.4 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 6293016 (600618 hex)
+Bytes transferred = 6293320 (600748 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67173 (10665 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x600618 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x600748 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013664
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc3-200087-g9e66317 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Oct 1 15:36:44 PDT 2017
+Linux version 4.14.0-rc4-200558-g8a5776a (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Oct 8 21:37:06 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
       .data : 0xc0f00000 - 0xc0fc4920   ( 787 kB)
-       .bss : 0xc0fce508 - 0xc10196bc   ( 301 kB)
+       .bss : 0xc0fce514 - 0xc10196bc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -537,7 +537,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:15 UTC (946684815)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:41 UTC (946684961)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-10-08 21:44:31.608559835 -0700
+++ /tmp/foob	2017-10-08 21:44:31.612559852 -0700
@@ -43,7 +43,7 @@
 	 #################################################################
 	 #################################################################
 	 ####################################
-	 2.9 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 8159752 (7c8208 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
@@ -52,7 +52,7 @@
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 68203 (10a6b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc3-200087-g9e66317 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Oct 1 15:36:12 PDT 2017
+Linux version 4.14.0-rc4-200558-g8a5776a (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Oct 8 21:37:02 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994372K/2097148K available (10240K kernel code, 1631K rwdata, 4500K rodata, 2048K init, 405K bss, 37240K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1994372K/2097148K available (10240K kernel code, 1631K rwdata, 4504K rodata, 2048K init, 405K bss, 37240K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -413,10 +413,11 @@
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
+elan_i2c 1-0015: failed to read report data: -121
+elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
-elan_i2c 1-0015: failed to read report data: -121
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
@@ -445,7 +446,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-07 07:30:07 UTC (947230207)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-10-08 21:44:31.628559922 -0700
+++ /tmp/foob	2017-10-08 21:44:31.632559939 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 2.9 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 6293016 (600618 hex)
+Bytes transferred = 6293320 (600748 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.7 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 68203 (10a6b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x600618 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x600748 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a6a
@@ -63,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc3-200087-g9e66317 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Oct 1 15:36:44 PDT 2017
+Linux version 4.14.0-rc4-200558-g8a5776a (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Oct 8 21:37:06 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -99,7 +99,7 @@
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
       .data : 0xc0f00000 - 0xc0fc4920   ( 787 kB)
-       .bss : 0xc0fce508 - 0xc10196bc   ( 301 kB)
+       .bss : 0xc0fce514 - 0xc10196bc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -388,8 +388,8 @@
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 can: controller area network core (rev 20170425 abi 9)
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 NET: Registered protocol family 29
 can: raw protocol (rev 20170425)
 can: broadcast manager protocol (rev 20170425 t)
@@ -458,7 +458,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-07 07:30:44 UTC (947230244)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -479,5 +479,5 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.500971] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+root@(none):/# [    3.490986] usb 1-3: new high-speed USB device number 3 using tegra-xusb
 asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-10-08 21:44:31.652560025 -0700
+++ /tmp/foob	2017-10-08 21:44:31.656560042 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801076 us
+Bootloader-Cpu Init at (time stamp): 1801084 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801050  us
-Bootloader (AosInit) time: 734227 us
-Bootloader Main Init time: 188253 us
-BlockDevMgrInit time: 157644 us
+Miscellaneous avp time: 1801058  us
+Bootloader (AosInit) time: 734239 us
+Bootloader Main Init time: 180081 us
+BlockDevMgrInit time: 157541 us
 PartMgrInit time: 8 us
 StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 76829 us 
-Display init time: 574719 us
+Partition Table load: 76644 us 
+Display init time: 574951 us
 Kernel read time: 1345 msec
 Load Kernel Size:   17825792 Bytes
 Bl Kernel ReadSpeed: 13253000 Bytes/sec
-Pre-Kernel Jump time: 3836814 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 8808821 us
+Pre-Kernel Jump time: 3839144 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 8803185 us
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc3-200087-g9e66317 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Oct 1 15:39:11 PDT 2017
+Linux version 4.14.0-rc4-318799-g8a5776a5f498 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Oct 8 21:42:18 PDT 2017
 Boot CPU: AArch64 Processor [4e0f0000]
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
@@ -200,7 +200,7 @@
 Freeing initrd memory: 1284K
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.979:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.987:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -289,8 +289,8 @@
 +5V_HDMI_CON: disabling
 ALSA device list:
   No soundcards found.
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 Freeing unused kernel memory: 1152K
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
 
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-10-08 21:44:31.660560060 -0700
+++ /tmp/foob	2017-10-08 21:44:31.664560078 -0700
@@ -48,7 +48,7 @@
 	 #################################################################
 	 #################################################################
 	 ####################################
-	 3.4 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 8159752 (7c8208 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc3-200087-g9e66317 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Oct 1 15:36:12 PDT 2017
+Linux version 4.14.0-rc4-200558-g8a5776a (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Oct 8 21:37:02 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -93,7 +93,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 954308K/1048576K available (10240K kernel code, 1631K rwdata, 4500K rodata, 2048K init, 405K bss, 28732K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 954308K/1048576K available (10240K kernel code, 1631K rwdata, 4504K rodata, 2048K init, 405K bss, 28732K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-10-08 21:44:31.680560147 -0700
+++ /tmp/foob	2017-10-08 21:44:31.684560164 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -46,9 +46,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3.4 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6293016 (600618 hex)
+Bytes transferred = 6293320 (600748 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x600618 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x600748 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc3-200087-g9e66317 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Oct 1 15:36:44 PDT 2017
+Linux version 4.14.0-rc4-200558-g8a5776a (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Oct 8 21:37:06 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -102,7 +102,7 @@
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
       .data : 0xc0f00000 - 0xc0fc4920   ( 787 kB)
-       .bss : 0xc0fce508 - 0xc10196bc   ( 301 kB)
+       .bss : 0xc0fce514 - 0xc10196bc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
 	Tasks RCU enabled.
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-10-08 21:44:31.704560250 -0700
+++ /tmp/foob	2017-10-08 21:44:31.708560268 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 815222 us
-Basic modules init took 349151 us
-Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14368 us
-Carveout took 253944 us
-CPU initialization took 92479 us
-Total time taken by TegraBoot 709942 us
+Tegraboot started after 732450 us
+Basic modules init took 350359 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14388 us
+Carveout took 252668 us
+CPU initialization took 92456 us
+Total time taken by TegraBoot 709871 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.654] LPDDR4 Training: Number of tables = 10
+64b[0001.573] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16505344 bytes read in 835 ms (18.9 MiB/s)
+16505344 bytes read in 830 ms (19 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-45170 bytes read in 452 ms (96.7 KiB/s)
+45170 bytes read in 453 ms (96.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc3-200087-g9e66317 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Oct 1 15:39:11 PDT 2017
+Linux version 4.14.0-rc4-318799-g8a5776a5f498 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Oct 8 21:42:18 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-10-08 21:44:31.724560337 -0700
+++ /tmp/foob	2017-10-08 21:44:31.724560337 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc3-200087-g9e66317 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Oct 1 15:39:11 PDT 2017
+Linux version 4.14.0-rc4-318799-g8a5776a5f498 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Oct 8 21:42:18 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
@@ -252,16 +252,16 @@
 Key type dns_resolver registered
 registered taskstats version 1
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+input: gpio-keys as /devices/platform/gpio-keys/input/input0
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
+max77686-rtc max77620-rtc: setting system clock to 2001-07-20 10:06:37 UTC (995623597)
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
-input: gpio-keys as /devices/platform/gpio-keys/input/input0
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
-max77686-rtc max77620-rtc: setting system clock to 2001-07-13 04:03:27 UTC (994997007)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-10-08 21:44:31.732560372 -0700
+++ /tmp/foob	2017-10-08 21:44:31.736560389 -0700
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 ####################################
-	 2.8 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 8159752 (7c8208 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc3-200087-g9e66317 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Oct 1 15:36:12 PDT 2017
+Linux version 4.14.0-rc4-200558-g8a5776a (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Oct 8 21:37:02 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -85,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993396K/2096128K available (10240K kernel code, 1631K rwdata, 4500K rodata, 2048K init, 405K bss, 37196K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993396K/2096128K available (10240K kernel code, 1631K rwdata, 4504K rodata, 2048K init, 405K bss, 37196K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -473,4 +473,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.544055] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.543695] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-10-08 21:44:31.756560476 -0700
+++ /tmp/foob	2017-10-08 21:44:31.760560493 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6293016 (600618 hex)
+Bytes transferred = 6293320 (600748 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.7 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51860 (ca94 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x600618 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x600748 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa93 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc3-200087-g9e66317 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Oct 1 15:36:44 PDT 2017
+Linux version 4.14.0-rc4-200558-g8a5776a (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Oct 8 21:37:06 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -94,7 +94,7 @@
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
       .data : 0xc0f00000 - 0xc0fc4920   ( 787 kB)
-       .bss : 0xc0fce508 - 0xc10196bc   ( 301 kB)
+       .bss : 0xc0fce514 - 0xc10196bc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
