Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:08.908002371 -0700
+++ /tmp/foob	2016-06-19 22:12:08.912002372 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Jun 12 07:36:41 PDT 2016
+Linux version 4.7.0-rc4+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Jun 19 22:07:36 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -19,15 +19,15 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffff800039c00000-ffff80003dbfffff]
-Memory: 935196K/1048576K available (7492K kernel code, 647K rwdata, 3448K rodata, 960K init, 259K bss, 96996K reserved, 16384K cma-reserved)
+Memory: 935196K/1048576K available (7492K kernel code, 646K rwdata, 3448K rodata, 960K init, 260K bss, 96996K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087c0000   (  7424 KB)
     .rodata : 0xffff0000087c0000 - 0xffff000008b30000   (  3520 KB)
       .init : 0xffff000008b30000 - 0xffff000008c20000   (   960 KB)
-      .data : 0xffff000008c20000 - 0xffff000008cc1e00   (   648 KB)
-       .bss : 0xffff000008cc1e00 - 0xffff000008d02ac0   (   260 KB)
+      .data : 0xffff000008c20000 - 0xffff000008cc1a00   (   647 KB)
+       .bss : 0xffff000008cc1a00 - 0xffff000008d02ac0   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -98,7 +98,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.592:1): initialized
+audit: type=2000 audit(0.584:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -159,7 +159,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-06-12 14:37:14 UTC (1465742234)
+rtc-pl031 9010000.pl031: setting system clock to 2016-06-20 05:08:13 UTC (1466399293)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:08.948002383 -0700
+++ /tmp/foob	2016-06-19 22:12:08.948002383 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 7239024 (6e7570 hex)
+Bytes transferred = 7239360 (6e76c0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6e7570 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6e76c0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-95766-g5edb564 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jun 12 07:34:14 PDT 2016
+Linux version 4.7.0-rc4-96064-g33688ab (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jun 19 22:04:28 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10ad5bc   (14998 kB)
+      .text : 0xc0208000 - 0xc10ad50c   (14998 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141dba0   (1143 kB)
        .bss : 0xc141f000 - 0xc1474ca0   ( 344 kB)
@@ -259,30 +259,6 @@
 ehci-exynos: EHCI EXYNOS driver
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
-Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.7.0-rc3-95766-g5edb564 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030f9d0>] (unwind_backtrace) from [<c030b0c0>] (show_stack+0x10/0x14)
-[<c030b0c0>] (show_stack) from [<c0585a88>] (dump_stack+0x84/0xa4)
-[<c0585a88>] (dump_stack) from [<c0341b5c>] (__warn+0xd4/0x100)
-[<c0341b5c>] (__warn) from [<c0341ba8>] (warn_slowpath_null+0x20/0x28)
-[<c0341ba8>] (warn_slowpath_null) from [<c06a0030>] (__of_reset_control_get+0x1bc/0x228)
-[<c06a0030>] (__of_reset_control_get) from [<c091236c>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c091236c>] (tegra_ehci_probe) from [<c077945c>] (platform_drv_probe+0x50/0xb0)
-[<c077945c>] (platform_drv_probe) from [<c0777dc0>] (driver_probe_device+0x1f4/0x2b0)
-[<c0777dc0>] (driver_probe_device) from [<c0777f28>] (__driver_attach+0xac/0xb0)
-[<c0777f28>] (__driver_attach) from [<c07763d4>] (bus_for_each_dev+0x54/0x88)
-[<c07763d4>] (bus_for_each_dev) from [<c07773b4>] (bus_add_driver+0xe8/0x1f4)
-[<c07773b4>] (bus_add_driver) from [<c0778744>] (driver_register+0x78/0xf4)
-[<c0778744>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
-[<c0301e7c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5bba4>] (kernel_init+0x8/0x118)
-[<c0b5bba4>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
----[ end trace 05ec5bae1179d782 ]---
-tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
-tegra-ehci 7d008000.usb: continuing, but please update your DT
 ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
 ohci-pci: OHCI PCI platform driver
 ohci-platform: OHCI generic platform driver
@@ -326,9 +302,9 @@
 usb2_vbus: supplied by vdd-sys-5v0
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
+[drm] No driver support for vblank timestamp query.
 mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
-[drm] No driver support for vblank timestamp query.
  mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
@@ -346,7 +322,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-06 01:54:58 UTC (954986098)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-13 16:25:36 UTC (955643136)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:08.956002386 -0700
+++ /tmp/foob	2016-06-19 22:12:08.956002386 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 #####################################################
-	 2.8 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5541904 (549010 hex)
+Bytes transferred = 5545488 (549e10 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x549010 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x549e10 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-95766-g5edb564 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jun 12 07:34:20 PDT 2016
+Linux version 4.7.0-rc4-96064-g33688ab (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jun 19 22:04:38 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5be00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5be40, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000976K/2096128K available (7416K kernel code, 374K rwdata, 2820K rodata, 1024K init, 366K bss, 29616K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999952K/2096128K available (7416K kernel code, 374K rwdata, 2824K rodata, 1024K init, 366K bss, 30640K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0aff0e0   (11229 kB)
-      .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5d980   ( 375 kB)
-       .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
+      .text : 0xc0008000 - 0xc0b00108   (11233 kB)
+      .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
+      .data : 0xc0d00000 - 0xc0d5d9c0   ( 375 kB)
+       .bss : 0xc0d5f000 - 0xc0dbab18   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -249,30 +249,6 @@
 ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
 ehci-pci: EHCI PCI platform driver
 tegra-ehci: Tegra EHCI driver
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
-Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.7.0-rc3-95766-g5edb564 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033d554>] (dump_stack+0x84/0xa4)
-[<c033d554>] (dump_stack) from [<c011f2f4>] (__warn+0xd4/0x100)
-[<c011f2f4>] (__warn) from [<c011f340>] (warn_slowpath_null+0x20/0x28)
-[<c011f340>] (warn_slowpath_null) from [<c03c0b38>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0b38>] (__of_reset_control_get) from [<c0514234>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0514234>] (tegra_ehci_probe) from [<c0441768>] (platform_drv_probe+0x50/0xb0)
-[<c0441768>] (platform_drv_probe) from [<c0440194>] (driver_probe_device+0x1f4/0x2b0)
-[<c0440194>] (driver_probe_device) from [<c04402fc>] (__driver_attach+0xac/0xb0)
-[<c04402fc>] (__driver_attach) from [<c043e7b8>] (bus_for_each_dev+0x54/0x88)
-[<c043e7b8>] (bus_for_each_dev) from [<c043f788>] (bus_add_driver+0xe8/0x1f4)
-[<c043f788>] (bus_add_driver) from [<c0440af8>] (driver_register+0x78/0xf4)
-[<c0440af8>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
-[<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f1c58>] (kernel_init+0x8/0x118)
-[<c07f1c58>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 285a2b1832f51c98 ]---
-tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
-tegra-ehci 7d008000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
 cdc_acm: USB Abstract Control Model driver for USB modems and ISDN adapters
 usbcore: registered new interface driver cdc_wdm
@@ -299,10 +275,10 @@
 mmc1: SDHCI controller on 78000600.sdhci [78000600.sdhci] using ADMA
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
+mmc0: new high speed SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
@@ -344,7 +320,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-06 01:57:28 UTC (954986248)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-13 16:28:07 UTC (955643287)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -353,10 +329,10 @@
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
+Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in th[    5.391856] usb 1-1: new high-speed USB device number 2 using tegra-ehci
-is shell
+bash: no j[    5.282120] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+ob control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:08.964002389 -0700
+++ /tmp/foob	2016-06-19 22:12:08.964002389 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -34,20 +34,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 4 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 7239024 (6e7570 hex)
+Bytes transferred = 7239360 (6e76c0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.7 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6e7570 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6e76c0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-95766-g5edb564 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jun 12 07:34:14 PDT 2016
+Linux version 4.7.0-rc4-96064-g33688ab (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jun 19 22:04:28 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -85,7 +85,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10ad5bc   (14998 kB)
+      .text : 0xc0208000 - 0xc10ad50c   (14998 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141dba0   (1143 kB)
        .bss : 0xc141f000 - 0xc1474ca0   ( 344 kB)
@@ -251,30 +251,6 @@
 ehci-exynos: EHCI EXYNOS driver
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
-Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.7.0-rc3-95766-g5edb564 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030f9d0>] (unwind_backtrace) from [<c030b0c0>] (show_stack+0x10/0x14)
-[<c030b0c0>] (show_stack) from [<c0585a88>] (dump_stack+0x84/0xa4)
-[<c0585a88>] (dump_stack) from [<c0341b5c>] (__warn+0xd4/0x100)
-[<c0341b5c>] (__warn) from [<c0341ba8>] (warn_slowpath_null+0x20/0x28)
-[<c0341ba8>] (warn_slowpath_null) from [<c06a0030>] (__of_reset_control_get+0x1bc/0x228)
-[<c06a0030>] (__of_reset_control_get) from [<c091236c>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c091236c>] (tegra_ehci_probe) from [<c077945c>] (platform_drv_probe+0x50/0xb0)
-[<c077945c>] (platform_drv_probe) from [<c0777dc0>] (driver_probe_device+0x1f4/0x2b0)
-[<c0777dc0>] (driver_probe_device) from [<c0777f28>] (__driver_attach+0xac/0xb0)
-[<c0777f28>] (__driver_attach) from [<c07763d4>] (bus_for_each_dev+0x54/0x88)
-[<c07763d4>] (bus_for_each_dev) from [<c07773b4>] (bus_add_driver+0xe8/0x1f4)
-[<c07773b4>] (bus_add_driver) from [<c0778744>] (driver_register+0x78/0xf4)
-[<c0778744>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
-[<c0301e7c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5bba4>] (kernel_init+0x8/0x118)
-[<c0b5bba4>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
----[ end trace 15a07b315df47f5b ]---
-tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
-tegra-ehci 7d004000.usb: continuing, but please update your DT
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
 tegra-ehci 7d004000.usb: irq 352, io mem 0x7d004000
@@ -401,8 +377,8 @@
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
-ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
+ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
@@ -449,11 +425,8 @@
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
-cpu cpu1: cpufreq_init: failed to get clk: -2
-cpu cpu2: cpufreq_init: failed to get clk: -2
-cpu cpu3: cpufreq_init: failed to get clk: -2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:12 UTC (946684812)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:06:00 UTC (946685160)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:08.972002392 -0700
+++ /tmp/foob	2016-06-19 22:12:08.976002393 -0700
@@ -32,20 +32,20 @@
 	 #################################################################
 	 #################################################################
 	 #####################################################
-	 3.6 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 5541904 (549010 hex)
+Bytes transferred = 5545488 (549e10 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3 MiB/s
+	 3.7 MiB/s
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x549010 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x549e10 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-95766-g5edb564 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jun 12 07:34:20 PDT 2016
+Linux version 4.7.0-rc4-96064-g33688ab (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jun 19 22:04:38 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5be00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5be40, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -77,7 +77,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000856K/2096128K available (7416K kernel code, 374K rwdata, 2820K rodata, 1024K init, 366K bss, 29736K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999832K/2096128K available (7416K kernel code, 374K rwdata, 2824K rodata, 1024K init, 366K bss, 30760K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -85,10 +85,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0aff0e0   (11229 kB)
-      .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5d980   ( 375 kB)
-       .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
+      .text : 0xc0008000 - 0xc0b00108   (11233 kB)
+      .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
+      .data : 0xc0d00000 - 0xc0d5d9c0   ( 375 kB)
+       .bss : 0xc0d5f000 - 0xc0dbab18   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -241,30 +241,6 @@
 ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
 ehci-pci: EHCI PCI platform driver
 tegra-ehci: Tegra EHCI driver
-------------[ cut here ]------------
-WARNING: CPU: 2 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
-Modules linked in:
-CPU: 2 PID: 1 Comm: swapper/0 Not tainted 4.7.0-rc3-95766-g5edb564 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033d554>] (dump_stack+0x84/0xa4)
-[<c033d554>] (dump_stack) from [<c011f2f4>] (__warn+0xd4/0x100)
-[<c011f2f4>] (__warn) from [<c011f340>] (warn_slowpath_null+0x20/0x28)
-[<c011f340>] (warn_slowpath_null) from [<c03c0b38>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0b38>] (__of_reset_control_get) from [<c0514234>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0514234>] (tegra_ehci_probe) from [<c0441768>] (platform_drv_probe+0x50/0xb0)
-[<c0441768>] (platform_drv_probe) from [<c0440194>] (driver_probe_device+0x1f4/0x2b0)
-[<c0440194>] (driver_probe_device) from [<c04402fc>] (__driver_attach+0xac/0xb0)
-[<c04402fc>] (__driver_attach) from [<c043e7b8>] (bus_for_each_dev+0x54/0x88)
-[<c043e7b8>] (bus_for_each_dev) from [<c043f788>] (bus_add_driver+0xe8/0x1f4)
-[<c043f788>] (bus_add_driver) from [<c0440af8>] (driver_register+0x78/0xf4)
-[<c0440af8>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
-[<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f1c58>] (kernel_init+0x8/0x118)
-[<c07f1c58>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 3c9f9c34b1b769d3 ]---
-tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
-tegra-ehci 7d004000.usb: continuing, but please update your DT
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
 tegra-ehci 7d004000.usb: irq 352, io mem 0x7d004000
@@ -424,18 +400,18 @@
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 mmc1: new high speed SDHC card at address aaaa
 PCI: bus2: Fast back to back transfers disabled
-pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
+pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
 pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
- mmcblk1: p1 p2 p3
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
 pci 0000:02:00.0: BAR 2: assigned [mem 0x13100000-0x13100fff 64bit]
+ mmcblk1: p1 p2 p3
 pci 0000:02:00.0: BAR 0: assigned [io  0x1000-0x10ff]
 pci 0000:00:02.0: PCI bridge to [bus 02]
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
@@ -464,11 +440,8 @@
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
-cpu cpu1: cpufreq_init: failed to get clk: -2
-cpu cpu2: cpufreq_init: failed to get clk: -2
-cpu cpu3: cpufreq_init: failed to get clk: -2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:08 UTC (946684928)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:04:04 UTC (946685044)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -485,7 +458,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
+Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:08.980002394 -0700
+++ /tmp/foob	2016-06-19 22:12:08.984002395 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 2.8 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 7239024 (6e7570 hex)
+Bytes transferred = 7239360 (6e76c0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.7 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 66901 (10555 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6e7570 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6e76c0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013554
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-95766-g5edb564 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jun 12 07:34:14 PDT 2016
+Linux version 4.7.0-rc4-96064-g33688ab (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jun 19 22:04:28 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -93,7 +93,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10ad5bc   (14998 kB)
+      .text : 0xc0208000 - 0xc10ad50c   (14998 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141dba0   (1143 kB)
        .bss : 0xc141f000 - 0xc1474ca0   ( 344 kB)
@@ -329,15 +329,15 @@
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
-mmc2: Invalid maximum block size, assuming 512 bytes
 mmc0: new high speed MMC card at address 0001
+mmc2: Invalid maximum block size, assuming 512 bytes
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
+mmc1: new high speed SDIO card at address 0001
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 mmc2: new high speed SDHC card at address 0007
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:08.996002399 -0700
+++ /tmp/foob	2016-06-19 22:12:09.000002401 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #####################################################
-	 3.5 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5541904 (549010 hex)
+Bytes transferred = 5545488 (549e10 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 66901 (10555 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x549010 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x549e10 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013554
@@ -142,14 +142,14 @@
 max98090 0-0010: MAX98090 REVID=0x43
 max98090 0-0010: use default 2.8v micbias
 mmc0: new high speed MMC card at address 0001
-tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
-input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
+input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
+input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 NET: Registered protocol family 10
 mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
@@ -159,8 +159,8 @@
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 can: netlink gateway (rev 20130117) max_hops=1
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
@@ -243,10 +243,10 @@
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
 devtmpfs: mounted
-Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
+Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.814654] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+root@(none):/# [    4.774718] usb 1-3: new high-speed USB device number 3 using tegra-xusb
 asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
 usb 1-1: new high-speed USB device number 4 using tegra-xusb
 usb-storage 1-1:1.0: USB Mass Storage device detected
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:09.016002406 -0700
+++ /tmp/foob	2016-06-19 22:12:09.016002406 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801372 us
+Bootloader-Cpu Init at (time stamp): 1801017 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801346  us
-Bootloader (AosInit) time: 733453 us
-Bootloader Main Init time: 185486 us
-BlockDevMgrInit time: 157671 us
-PartMgrInit time: 9 us
-StorMgrInit time: 30 us
+Miscellaneous avp time: 1800991  us
+Bootloader (AosInit) time: 734248 us
+Bootloader Main Init time: 180078 us
+BlockDevMgrInit time: 157564 us
+PartMgrInit time: 8 us
+StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 76962 us 
-Display init time: 574383 us
-Kernel read time: 353 msec
+Partition Table load: 76670 us 
+Display init time: 574682 us
+Kernel read time: 352 msec
 Load Kernel Size:   15280128 Bytes
-Bl Kernel ReadSpeed: 43286000 Bytes/sec
-Pre-Kernel Jump time: 1657297 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5736269 us
+Bl Kernel ReadSpeed: 43409000 Bytes/sec
+Pre-Kernel Jump time: 1657581 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5730674 us
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Jun 12 07:36:41 PDT 2016
+Linux version 4.7.0-rc4+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Jun 19 22:07:36 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -92,15 +92,15 @@
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
 software IO TLB [mem 0xe5200000-0xe9200000] (64MB) mapped at [ffff800065200000-ffff8000691fffff]
-Memory: 1641788K/1775616K available (7492K kernel code, 647K rwdata, 3448K rodata, 960K init, 259K bss, 117444K reserved, 16384K cma-reserved)
+Memory: 1641788K/1775616K available (7492K kernel code, 646K rwdata, 3448K rodata, 960K init, 260K bss, 117444K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087c0000   (  7424 KB)
     .rodata : 0xffff0000087c0000 - 0xffff000008b30000   (  3520 KB)
       .init : 0xffff000008b30000 - 0xffff000008c20000   (   960 KB)
-      .data : 0xffff000008c20000 - 0xffff000008cc1e00   (   648 KB)
-       .bss : 0xffff000008cc1e00 - 0xffff000008d02ac0   (   260 KB)
+      .data : 0xffff000008c20000 - 0xffff000008cc1a00   (   647 KB)
+       .bss : 0xffff000008cc1a00 - 0xffff000008d02ac0   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -243,8 +243,8 @@
 9pnet: Installing 9P2000 support
 Key type dns_resolver registered
 registered taskstats version 1
-mmc0: new high speed MMC card at address 0001
 +VDD_LED: supplied by +VDD_MUX
+mmc0: new high speed MMC card at address 0001
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:09.032002411 -0700
+++ /tmp/foob	2016-06-19 22:12:09.036002413 -0700
@@ -49,7 +49,7 @@
 	 #######################################
 	 4 MiB/s
 done
-Bytes transferred = 7239024 (6e7570 hex)
+Bytes transferred = 7239360 (6e76c0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6e7570 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6e76c0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-95766-g5edb564 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jun 12 07:34:14 PDT 2016
+Linux version 4.7.0-rc4-96064-g33688ab (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jun 19 22:04:28 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10ad5bc   (14998 kB)
+      .text : 0xc0208000 - 0xc10ad50c   (14998 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141dba0   (1143 kB)
        .bss : 0xc141f000 - 0xc1474ca0   ( 344 kB)
@@ -270,30 +270,6 @@
 ehci-exynos: EHCI EXYNOS driver
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
-------------[ cut here ]------------
-WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
-Modules linked in:
-CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.7.0-rc3-95766-g5edb564 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030f9d0>] (unwind_backtrace) from [<c030b0c0>] (show_stack+0x10/0x14)
-[<c030b0c0>] (show_stack) from [<c0585a88>] (dump_stack+0x84/0xa4)
-[<c0585a88>] (dump_stack) from [<c0341b5c>] (__warn+0xd4/0x100)
-[<c0341b5c>] (__warn) from [<c0341ba8>] (warn_slowpath_null+0x20/0x28)
-[<c0341ba8>] (warn_slowpath_null) from [<c06a0030>] (__of_reset_control_get+0x1bc/0x228)
-[<c06a0030>] (__of_reset_control_get) from [<c091236c>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c091236c>] (tegra_ehci_probe) from [<c077945c>] (platform_drv_probe+0x50/0xb0)
-[<c077945c>] (platform_drv_probe) from [<c0777dc0>] (driver_probe_device+0x1f4/0x2b0)
-[<c0777dc0>] (driver_probe_device) from [<c0777f28>] (__driver_attach+0xac/0xb0)
-[<c0777f28>] (__driver_attach) from [<c07763d4>] (bus_for_each_dev+0x54/0x88)
-[<c07763d4>] (bus_for_each_dev) from [<c07773b4>] (bus_add_driver+0xe8/0x1f4)
-[<c07773b4>] (bus_add_driver) from [<c0778744>] (driver_register+0x78/0xf4)
-[<c0778744>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
-[<c0301e7c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5bba4>] (kernel_init+0x8/0x118)
-[<c0b5bba4>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
----[ end trace 2c6b76aa884e384d ]---
-tegra-ehci c5000000.usb: can't get utmi-pads reset from the PHY
-tegra-ehci c5000000.usb: continuing, but please update your DT
 ulpi_phy_power_on: ulpi write failed
 tegra-ehci c5004000.usb: Failed to power on the phy
 tegra-ehci: probe of c5004000.usb failed with error -110
@@ -344,20 +320,21 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-mmc0: host does not support reading read-only switch, assuming write-enable
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-mmc1: new high speed SD card at address b368
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
+mmc0: host does not support reading read-only switch, assuming write-enable
+Waiting for root device PARTUUID=11eb23c1-01...
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 EXT4-fs (mmcblk0p1): recovery complete
@@ -365,7 +342,7 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:09.044002415 -0700
+++ /tmp/foob	2016-06-19 22:12:09.048002416 -0700
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 #####################################################
-	 905.3 KiB/s
+	 906.3 KiB/s
 done
-Bytes transferred = 5541904 (549010 hex)
+Bytes transferred = 5545488 (549e10 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x549010 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x549e10 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-95766-g5edb564 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jun 12 07:34:20 PDT 2016
+Linux version 4.7.0-rc4-96064-g33688ab (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jun 19 22:04:38 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0c5be00, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0d5be40, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 961792K/1048576K available (7416K kernel code, 374K rwdata, 2820K rodata, 1024K init, 366K bss, 21248K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 960768K/1048576K available (7416K kernel code, 374K rwdata, 2824K rodata, 1024K init, 366K bss, 22272K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,10 +97,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0aff0e0   (11229 kB)
-      .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5d980   ( 375 kB)
-       .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
+      .text : 0xc0008000 - 0xc0b00108   (11233 kB)
+      .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
+      .data : 0xc0d00000 - 0xc0d5d9c0   ( 375 kB)
+       .bss : 0xc0d5f000 - 0xc0dbab18   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -262,30 +262,6 @@
 ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
 ehci-pci: EHCI PCI platform driver
 tegra-ehci: Tegra EHCI driver
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
-Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.7.0-rc3-95766-g5edb564 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033d554>] (dump_stack+0x84/0xa4)
-[<c033d554>] (dump_stack) from [<c011f2f4>] (__warn+0xd4/0x100)
-[<c011f2f4>] (__warn) from [<c011f340>] (warn_slowpath_null+0x20/0x28)
-[<c011f340>] (warn_slowpath_null) from [<c03c0b38>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0b38>] (__of_reset_control_get) from [<c0514234>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0514234>] (tegra_ehci_probe) from [<c0441768>] (platform_drv_probe+0x50/0xb0)
-[<c0441768>] (platform_drv_probe) from [<c0440194>] (driver_probe_device+0x1f4/0x2b0)
-[<c0440194>] (driver_probe_device) from [<c04402fc>] (__driver_attach+0xac/0xb0)
-[<c04402fc>] (__driver_attach) from [<c043e7b8>] (bus_for_each_dev+0x54/0x88)
-[<c043e7b8>] (bus_for_each_dev) from [<c043f788>] (bus_add_driver+0xe8/0x1f4)
-[<c043f788>] (bus_add_driver) from [<c0440af8>] (driver_register+0x78/0xf4)
-[<c0440af8>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
-[<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f1c58>] (kernel_init+0x8/0x118)
-[<c07f1c58>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace c9c7c4f0258a4b92 ]---
-tegra-ehci c5000000.usb: can't get utmi-pads reset from the PHY
-tegra-ehci c5000000.usb: continuing, but please update your DT
 ulpi_phy_power_on: ulpi write failed
 tegra-ehci c5004000.usb: Failed to power on the phy
 tegra-ehci: probe of c5004000.usb failed with error -110
@@ -344,11 +320,11 @@
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
 can: netlink gateway (rev 20130117) max_hops=1
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
@@ -373,7 +349,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
-Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
+Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:09.068002423 -0700
+++ /tmp/foob	2016-06-19 22:12:09.068002423 -0700
@@ -1,7 +1,7 @@
 [TegraBoot] (version 23.00.2015.14-mobile-1ef66670)
 Processing in cold boot mode Bootloader 2
 A01Q Bootrom Patch rev = 3
-Power-up reason: pmu watchdog
+Power-up reason: on button
 No Battery Present
 Platform has Ddr4 type ram
 max77620 disabling SD1 Remote Sense
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 730791 us
-Basic modules init took 348822 us
+Tegraboot started after 766198 us
+Basic modules init took 348779 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14382 us
-Carveout took 252729 us
-CPU initialization took 92312 us
-Total time taken by TegraBoot 708245 us
+Next stage binary read took 14365 us
+Carveout took 252842 us
+CPU initialization took 92327 us
+Total time taken by TegraBoot 708313 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.570] LPDDR4 Training: Number of tables = 10
+64b[0001.605] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-12852736 bytes read in 754 ms (16.3 MiB/s)
+12851712 bytes read in 753 ms (16.3 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-37108 bytes read in 454 ms (79.1 KiB/s)
+37108 bytes read in 453 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 496 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Jun 12 07:36:41 PDT 2016
+Linux version 4.7.0-rc4+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Jun 19 22:07:36 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -166,15 +166,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2980144K/3131392K available (7492K kernel code, 647K rwdata, 3448K rodata, 960K init, 259K bss, 134864K reserved, 16384K cma-reserved)
+Memory: 2980144K/3131392K available (7492K kernel code, 646K rwdata, 3448K rodata, 960K init, 260K bss, 134864K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087c0000   (  7424 KB)
     .rodata : 0xffff0000087c0000 - 0xffff000008b30000   (  3520 KB)
       .init : 0xffff000008b30000 - 0xffff000008c20000   (   960 KB)
-      .data : 0xffff000008c20000 - 0xffff000008cc1e00   (   648 KB)
-       .bss : 0xffff000008cc1e00 - 0xffff000008d02ac0   (   260 KB)
+      .data : 0xffff000008c20000 - 0xffff000008cc1a00   (   647 KB)
+       .bss : 0xffff000008cc1a00 - 0xffff000008d02ac0   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -264,7 +264,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.455:1): initialized
+audit: type=2000 audit(0.451:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:09.088002429 -0700
+++ /tmp/foob	2016-06-19 22:12:09.092002430 -0700
@@ -1,69 +1,4 @@
-071]
-earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
-bootconsole [uart0] enabled
-efi: Getting EFI parameters from FDT:
-efi: UEFI not found.
-cma: Reserved 16 MiB at 0x00000000f5400000
-psci: probing for conduit method from DT.
-psci: PSCIv1.0 detected in firmware.
-psci: Using standard PSCI v0.2 function IDs
-psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 20 pages/cpu @ffff8000bff54000 s42752 r8192 d30976 u81920
-Detected PIPT I-cache on CPU0
-CPU features: enabling workaround for ARM erratum 832075
-CPU features: enabling workaround for ARM erratum 834220
-Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 735336
-Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
-log_buf_len individual max cpu contribution: 4096 bytes
-log_buf_len total cpu_extra contributions: 12288 bytes
-log_buf_len min size: 16384 bytes
-log_buf_len: 32768 bytes
-early log buf free: 14436(88%)
-PID hash table entries: 4096 (order: 3, 32768 bytes)
-Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
-Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
-software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2834380K/2988032K available (7492K kernel code, 647K rwdata, 3448K rodata, 960K init, 259K bss, 137268K reserved, 16384K cma-reserved)
-Virtual kernel memory layout:
-    modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
-    vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff0000087c0000   (  7424 KB)
-    .rodata : 0xffff0000087c0000 - 0xffff000008b30000   (  3520 KB)
-      .init : 0xffff000008b30000 - 0xffff000008c20000   (   960 KB)
-      .data : 0xffff000008c20000 - 0xffff000008cc1e00   (   648 KB)
-       .bss : 0xffff000008cc1e00 - 0xffff000008d02ac0   (   260 KB)
-    fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
-    PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
-    vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
-              0xffff7e0000000000 - 0xffff7e0003000000   (    48 MB actual)
-    memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
-SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
-Preemptible hierarchical RCU implementation.
-	Build-time adjustment of leaf fanout to 64.
-	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=4
-NR_IRQS:64 nr_irqs:64 0
-GIC: Using split EOI/Deactivate mode
-/interrupt-controller@60004000: 192 interrupts forwarded to /interrupt-controller@50041000
-boot misc0 0x0: expected 0x80000
- (comparison mask = 0x400ffffb)
-boot misc2 0x1f720005: expected 0x1f720f05
- (comparison mask = 0xffffff17)
-PLL_C already enabled. Postponing set full defaults
-PLL_MB already enabled. Postponing set full defaults
-PLL_U already enabled. Postponing set full defaults
-PLL_D already enabled. Postponing set full defaults
-PLL_DP already enabled. Postponing set full defaults
-boot misc0 0x0: expected 0x20
- (comparison mask = 0x7ffffff)
-boot misc1 0x90000000: expected 0xc0000000
- (comparison mask = 0xf8000000)
-boot misc2 0x0: expected 0xf400f0da
- (comparison mask = 0xffffffff)
-boot misc3 0xafb0: expected 0x2004f400
- (comparison mask = 0xffffffff)
-PLL_P already enabled. Postponing set full defaults
-PLL_X already enabled. Postponing set full defaults
+ng set full defaults
 boot misc1 0x0: expected 0x20
  (comparison mask = 0xffffff)
 Architected cp15 timer(s) running at 19.20MHz (phys).
@@ -73,7 +8,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Jun 12 07:36:41 PDT 2016
+Linux version 4.7.0-rc4+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Jun 19 22:07:36 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
 bootconsole [uart0] enabled
@@ -99,15 +34,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2834380K/2988032K available (7492K kernel code, 647K rwdata, 3448K rodata, 960K init, 259K bss, 137268K reserved, 16384K cma-reserved)
+Memory: 2834384K/2988032K available (7492K kernel code, 646K rwdata, 3448K rodata, 960K init, 260K bss, 137264K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087c0000   (  7424 KB)
     .rodata : 0xffff0000087c0000 - 0xffff000008b30000   (  3520 KB)
       .init : 0xffff000008b30000 - 0xffff000008c20000   (   960 KB)
-      .data : 0xffff000008c20000 - 0xffff000008cc1e00   (   648 KB)
-       .bss : 0xffff000008cc1e00 - 0xffff000008d02ac0   (   260 KB)
+      .data : 0xffff000008c20000 - 0xffff000008cc1a00   (   647 KB)
+       .bss : 0xffff000008cc1a00 - 0xffff000008d02ac0   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -136,7 +71,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0xafb0: expected 0x2004f400
+boot misc3 0xb800: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -206,7 +141,7 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 2364K (ffff800009ee8000 - ffff80000a137000)
+Freeing initrd memory: 2360K (ffff800009ee8000 - ffff80000a136000)
 kvm [1]: 8-bit VMID
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -214,7 +149,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.235:1): initialized
+audit: type=2000 audit(0.227:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:09.096002432 -0700
+++ /tmp/foob	2016-06-19 22:12:09.096002432 -0700
@@ -41,7 +41,7 @@
 	 #######################################
 	 3.2 MiB/s
 done
-Bytes transferred = 7239024 (6e7570 hex)
+Bytes transferred = 7239360 (6e76c0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6e7570 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6e76c0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-95766-g5edb564 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jun 12 07:34:14 PDT 2016
+Linux version 4.7.0-rc4-96064-g33688ab (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jun 19 22:04:28 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10ad5bc   (14998 kB)
+      .text : 0xc0208000 - 0xc10ad50c   (14998 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141dba0   (1143 kB)
        .bss : 0xc141f000 - 0xc1474ca0   ( 344 kB)
@@ -253,30 +253,6 @@
 ehci-exynos: EHCI EXYNOS driver
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
-Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.7.0-rc3-95766-g5edb564 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030f9d0>] (unwind_backtrace) from [<c030b0c0>] (show_stack+0x10/0x14)
-[<c030b0c0>] (show_stack) from [<c0585a88>] (dump_stack+0x84/0xa4)
-[<c0585a88>] (dump_stack) from [<c0341b5c>] (__warn+0xd4/0x100)
-[<c0341b5c>] (__warn) from [<c0341ba8>] (warn_slowpath_null+0x20/0x28)
-[<c0341ba8>] (warn_slowpath_null) from [<c06a0030>] (__of_reset_control_get+0x1bc/0x228)
-[<c06a0030>] (__of_reset_control_get) from [<c091236c>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c091236c>] (tegra_ehci_probe) from [<c077945c>] (platform_drv_probe+0x50/0xb0)
-[<c077945c>] (platform_drv_probe) from [<c0777dc0>] (driver_probe_device+0x1f4/0x2b0)
-[<c0777dc0>] (driver_probe_device) from [<c0777f28>] (__driver_attach+0xac/0xb0)
-[<c0777f28>] (__driver_attach) from [<c07763d4>] (bus_for_each_dev+0x54/0x88)
-[<c07763d4>] (bus_for_each_dev) from [<c07773b4>] (bus_add_driver+0xe8/0x1f4)
-[<c07773b4>] (bus_add_driver) from [<c0778744>] (driver_register+0x78/0xf4)
-[<c0778744>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
-[<c0301e7c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5bba4>] (kernel_init+0x8/0x118)
-[<c0b5bba4>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
----[ end trace 33c6703069bfd7dd ]---
-tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
-tegra-ehci 7d004000.usb: continuing, but please update your DT
 ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
 ohci-pci: OHCI PCI platform driver
 ohci-platform: OHCI generic platform driver
@@ -323,11 +299,11 @@
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: 2x3 configuration
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
+mmc0: error -110 whilst initialising SD card
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
-mmc0: error -110 whilst initialising SD card
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-19 22:12:09.104002435 -0700
+++ /tmp/foob	2016-06-19 22:12:09.104002435 -0700
@@ -39,18 +39,18 @@
 	 #####################################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5541904 (549010 hex)
+Bytes transferred = 5545488 (549e10 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.7 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x549010 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x549e10 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-95766-g5edb564 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jun 12 07:34:20 PDT 2016
+Linux version 4.7.0-rc4-96064-g33688ab (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jun 19 22:04:38 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5be00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5be40, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000896K/2096128K available (7416K kernel code, 374K rwdata, 2820K rodata, 1024K init, 366K bss, 29696K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999872K/2096128K available (7416K kernel code, 374K rwdata, 2824K rodata, 1024K init, 366K bss, 30720K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0aff0e0   (11229 kB)
-      .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5d980   ( 375 kB)
-       .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
+      .text : 0xc0008000 - 0xc0b00108   (11233 kB)
+      .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
+      .data : 0xc0d00000 - 0xc0d5d9c0   ( 375 kB)
+       .bss : 0xc0d5f000 - 0xc0dbab18   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -243,30 +243,6 @@
 ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
 ehci-pci: EHCI PCI platform driver
 tegra-ehci: Tegra EHCI driver
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
-Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.7.0-rc3-95766-g5edb564 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033d554>] (dump_stack+0x84/0xa4)
-[<c033d554>] (dump_stack) from [<c011f2f4>] (__warn+0xd4/0x100)
-[<c011f2f4>] (__warn) from [<c011f340>] (warn_slowpath_null+0x20/0x28)
-[<c011f340>] (warn_slowpath_null) from [<c03c0b38>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0b38>] (__of_reset_control_get) from [<c0514234>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0514234>] (tegra_ehci_probe) from [<c0441768>] (platform_drv_probe+0x50/0xb0)
-[<c0441768>] (platform_drv_probe) from [<c0440194>] (driver_probe_device+0x1f4/0x2b0)
-[<c0440194>] (driver_probe_device) from [<c04402fc>] (__driver_attach+0xac/0xb0)
-[<c04402fc>] (__driver_attach) from [<c043e7b8>] (bus_for_each_dev+0x54/0x88)
-[<c043e7b8>] (bus_for_each_dev) from [<c043f788>] (bus_add_driver+0xe8/0x1f4)
-[<c043f788>] (bus_add_driver) from [<c0440af8>] (driver_register+0x78/0xf4)
-[<c0440af8>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
-[<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f1c58>] (kernel_init+0x8/0x118)
-[<c07f1c58>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 8719e4fc49eb6a2a ]---
-tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
-tegra-ehci 7d004000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
 cdc_acm: USB Abstract Control Model driver for USB modems and ISDN adapters
 usbcore: registered new interface driver cdc_wdm
@@ -399,8 +375,8 @@
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
+Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 root@(none):/# 
\ No newline at end of file
